<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> Chromosomal instability and its impact on selection during cancer development | Khanh N. Dinh Lab </title> <meta name="author" content="Khanh N. Dinh"> <meta name="description" content="Mathematical and computational methods to extract the occurrence rates and selection coefficients of chromosomal instability from DNA-sequencing data"> <meta name="keywords" content="Computational-biology, Mathematical-modeling"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%E2%9A%9B%EF%B8%8F&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://dinhngockhanh.github.io/projects/project_CIN_inference/"> <script src="/assets/js/theme.js?9a0c749ec5240d9cda97bc72359a72c0"></script> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script>initTheme();</script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> Khanh N. Dinh Lab </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about </a> </li> <li class="nav-item "> <a class="nav-link" href="/research/">research </a> </li> <li class="nav-item "> <a class="nav-link" href="/publications/">publications </a> </li> <li class="nav-item "> <a class="nav-link" href="/repositories/">repositories </a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">cv </a> </li> <li class="nav-item "> <a class="nav-link" href="/alumni/">alumni </a> </li> <li class="nav-item "> <a class="nav-link" href="/people/">people </a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="ti ti-sun-moon" id="light-toggle-system"></i> <i class="ti ti-moon-filled" id="light-toggle-dark"></i> <i class="ti ti-sun-filled" id="light-toggle-light"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">Chromosomal instability and its impact on selection during cancer development</h1> <p class="post-description">Mathematical and computational methods to extract the occurrence rates and selection coefficients of chromosomal instability from DNA-sequencing data</p> </header> <article> <hr> <p>Cancer evolution has been studied extensively as a process of accumulating point mutations, which affect one or a few nucleotides at specific locations in the genome. However, some cancers have been known to be driven mostly by copy number aberrations (CNAs). These events can be inferred by examining the total and allele-specific copy numbers across the cancer genome. Chromosomal instability (CIN) is said to occur when a tumor exhibits a high number of CNAs.</p> <p>Recent developments in single-cell DNA-sequencing have provided an unparalleled view into the diversity and ongoing evolution within a tumor. Direct Library Development+ (DLP+), developed by the <a href="https://www.mskcc.org/research-areas/labs/sohrab-shah" rel="external nofollow noopener" target="_blank">Sohrab Shah Lab</a>, is capable of producing genomic information for tens of thousands of cells, without bias due to amplification or non-uniform coverage <a class="citation" href="#laks2019clonal">(Laks et al., 2019)</a>. Using DLP+, researchers have uncovered varying levels of CIN in different cancers, both between patients with the same tumor characteristic and across different cancer types.</p> <div class="row"> <div class="col-sm mt-3 mt-md-0"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/project_CIN_inference_DLP_1-480.webp 480w,/assets/img/project_CIN_inference_DLP_1-800.webp 800w,/assets/img/project_CIN_inference_DLP_1-1400.webp 1400w," type="image/webp" sizes="95vw"></source> <img src="/assets/img/project_CIN_inference_DLP_1.png" class="img-fluid rounded z-depth-1" width="100%" height="auto" title="DLP+ pipeline" loading="lazy" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> </div> <div class="caption"> DLP+ <a class="citation" href="#laks2019clonal">(Laks et al., 2019)</a> provides copy number (CN) information at the cell level. Left: Overview of DLP+ experimental and computational pipeline. Right: Total copy numbers (top) and minor allele fractions (bottom) across the genome in one cell from an ovarian tumor. </div> <p>By grouping cells with similar copy numbers into clones and tracking them over time, DLP+ data provides a picture of competition between different clones. In some experiments, some clones were observed expanding through time, while others became extinct. This indicates that selection plays an important role during tumor growth, and certain CNAs are preferred over others as cancer progresses <a class="citation" href="#salehi2021clonal">(Salehi et al., 2021)</a>.</p> <div class="row"> <div class="col-sm mt-3 mt-md-0"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/project_CIN_inference_DLP_2-480.webp 480w,/assets/img/project_CIN_inference_DLP_2-800.webp 800w,/assets/img/project_CIN_inference_DLP_2-1400.webp 1400w," type="image/webp" sizes="95vw"></source> <img src="/assets/img/project_CIN_inference_DLP_2.png" class="img-fluid rounded z-depth-1" width="100%" height="auto" title="DLP+ signatures" loading="lazy" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> </div> <div class="caption"> Evolution of clones with distinct copy numbers from a triple-negative breast tumor over time <a class="citation" href="#salehi2021clonal">(Salehi et al., 2021)</a>. Left: Phylogeny of cells based on their copy number profiles. Cells with similar copy numbers are grouped into a clone. Right: Temporal trajectory of clonal fractions in the cell population (= number of cells in the clone, divided by number of cells in total). </div> <p>Furthermore, applications of DLP+ have led to the realization that knock-outs of certain important genes, such as TP53 and BRCA1/2 in the mammary epithelium, are associated with a significant increase in CIN <a class="citation" href="#funnell2022single">(Funnell et al., 2022)</a>. This manifests in higher number of polyploid cells (resulting from whole-genome duplications) and more chromosome missegregations. These results explain the high numbers of CNAs that have been observed in ovarian and breast cancers.</p> <div class="row"> <div class="col-sm mt-3 mt-md-0"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/project_CIN_inference_DLP_3-480.webp 480w,/assets/img/project_CIN_inference_DLP_3-800.webp 800w,/assets/img/project_CIN_inference_DLP_3-1400.webp 1400w," type="image/webp" sizes="95vw"></source> <img src="/assets/img/project_CIN_inference_DLP_3.png" class="img-fluid rounded z-depth-1" width="100%" height="auto" title="DLP+ signatures" loading="lazy" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> </div> <div class="caption"> Copy number changes associated with key genes being turned off in mammary epithelial cell lines <a class="citation" href="#funnell2022single">(Funnell et al., 2022)</a>. Top: Total and allele-specific copy numbers in cells with TP53 knocked out (left) and both TP53 and BRCA1 knocked out (right). Bottom: Statistics of cell populations, depending on whether TP53, BRCA1 or BRCA2 are knocked out. </div> <hr> <p>We seek to systematically study how CIN arises and affects the selection landscape during tumor growth. This requires a mathematical framework that incorporates different CNA mechanisms that occur during CIN. We developed CINner, an algorithm to simulate how CIN arises and affects the selection landscape during cancer growth <a class="citation" href="#dinh2024cinner">(Dinh et al., 2025)</a>. CINner, available as an <a href="https://github.com/dinhngockhanh/CINner" rel="external nofollow noopener" target="_blank">R package on Github</a>, is designed to efficiently model a cell population undergoing different types of CNAs and mutations, which change the cell karyotypes and increase the clonal diversity.</p> <div class="row"> <div class="col-sm mt-3 mt-md-0"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/project_CIN_inference_CINner_1-480.webp 480w,/assets/img/project_CIN_inference_CINner_1-800.webp 800w,/assets/img/project_CIN_inference_CINner_1-1400.webp 1400w," type="image/webp" sizes="95vw"></source> <img src="/assets/img/project_CIN_inference_CINner_1.jpg" class="img-fluid rounded z-depth-1" width="100%" height="auto" title="example image" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> </div> <div class="caption"> Mathematical model underlying CINner. </div> <p>CINner currently supports different CNA mechanisms, including whole-genome duplications, whole-chromosome and chromosome-arm missegregations, focal amplifications and deletions. We included three different selection models, designed to quantify the fitness of chromosome-arm level CNAs or driver mutations, or both.</p> <div class="row"> <div class="col-sm mt-3 mt-md-0"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/project_CIN_inference_CINner_2-480.webp 480w,/assets/img/project_CIN_inference_CINner_2-800.webp 800w,/assets/img/project_CIN_inference_CINner_2-1400.webp 1400w," type="image/webp" sizes="95vw"></source> <img src="/assets/img/project_CIN_inference_CINner_2.jpg" class="img-fluid rounded z-depth-1" width="100%" height="auto" title="example image" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> </div> <div class="caption"> Schematics for the selection model of chromosome arms (left) and driver mutations (right). </div> <p>When applied to whole genome sequencing data across all cancers in The Cancer Genome Atlas (TCGA), CINner inferred selection parameters for individual chromosome arms. These selection parameters strongly correlate with the gene imbalance on each arm from <a href="https://www.cell.com/fulltext/S0092-8674(13)01287-7" rel="external nofollow noopener" target="_blank">Davoli et al.</a>, indicating that selection rates inferred from CINner are estimates for the combined effects of genes located on different genomic regions.</p> <div class="row"> <div class="col-sm mt-3 mt-md-0"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/project_CIN_inference_CINner_3-480.webp 480w,/assets/img/project_CIN_inference_CINner_3-800.webp 800w,/assets/img/project_CIN_inference_CINner_3-1400.webp 1400w," type="image/webp" sizes="95vw"></source> <img src="/assets/img/project_CIN_inference_CINner_3.jpg" class="img-fluid rounded z-depth-1" width="100%" height="auto" title="example image" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> </div> <div class="caption"> Top: Schematic for the inference and analysis of cancer type-specific chromosome-arm selection parameters. Bottom: Comparison of selection rates inferred by CINner (x axis) from pan-cancer TCGA data, against gene balance scores (y axis). </div> <p>The same inference routine can be applied for individual cancer types, for which CINner finds selection parameters that faithfully recreate the copy number landscapes observed in data. These parameters are inferred using samples without whole-genome duplication (WGD), from which chromosome arms can be classified as GAIN (if the selection rate &gt; 1, hence gains of these arms are advantageous) or LOSS (if the selection rate &lt; 1, in which case losses increase cell fitness). The count and mean selection rate across these inferred GAIN and LOSS arms predict cancer-specific WGD prevalence well. On the one hand, this confirms that WGD is an important event that remolds the selection landscape during cancer development, as has been observed experimentally. On the other hand, this reconfirms CINner’s ability to uncover biologically relevant parameters from sample cohorts of relatively small sizes.</p> <div class="row"> <div class="col-sm mt-3 mt-md-0"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/project_CIN_inference_CINner_4-480.webp 480w,/assets/img/project_CIN_inference_CINner_4-800.webp 800w,/assets/img/project_CIN_inference_CINner_4-1400.webp 1400w," type="image/webp" sizes="95vw"></source> <img src="/assets/img/project_CIN_inference_CINner_4.jpg" class="img-fluid rounded z-depth-1" width="100%" height="auto" title="example image" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> </div> <div class="caption"> Top: Comparison between gain/loss frequencies from data and CINner with inferred selection rates, for ovary adenocarcinoma. Bottom: Correlation between cancer-specific WGD proportion and count of GAIN arms (left) and mean selection rate of LOSS arms (right). </div> <p>In addition to its applications in assessing chromosomal instability in DNA-sequencing data, CINner can also be used to benchmark analysis pipelines. For instance, it was utilized in <a class="citation" href="#mcpherson2025ongoing">(McPherson et al., 2025)</a> to assess the accuracy of inferred CIN and WGD from DLP data.</p> <hr> <p>With CINner, we now have a framework to uncover selection parameters for individual genomic regions. However, we are also interested in finding the rates at which CNA mechanisms occur, such as chromosome missegregations. This requires more information than what bulk genomic data can offer. This is because if a particular CNA is frequently observed in data, it can be explained by a spectrum of parameters in CINner. On one extreme, it could be because the CNA occurrence is high, therefore the event takes place often across different patients even if it does not significantly impact cancer fitness. On the other extreme, it could be because the selection rate associated with the CNA event is high, in which case it takes a long time to emerge but always expands across the entire tumor when it does. The parameters therefore are confounded in bulk data alone.</p> <p>In <a class="citation" href="#xiang2024inference">(Xiang et al., 2024)</a>, we developed a parameter inference routine that employs both bulk data and information from single-cell (sc) sequencing, such as DLP+. The algorithm uses <a href="https://academic.oup.com/bioinformatics/article/35/10/1720/5132692" rel="external nofollow noopener" target="_blank">ABC-rf</a>, an Approximate Bayesian Computation (ABC) method with random forests that produces reliable posterior distributions with high tolerance for noise. The ABC framework summarizes both data and model simulations with statistics, for which we considered a wide range of measurements based on bulk CN, single-cell CN profiles, and the phylogeny for cells in the single-cell samples.</p> <div class="row"> <div class="col-sm mt-3 mt-md-0"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/project_CIN_inference_theory_1-480.webp 480w,/assets/img/project_CIN_inference_theory_1-800.webp 800w,/assets/img/project_CIN_inference_theory_1-1400.webp 1400w," type="image/webp" sizes="95vw"></source> <img src="/assets/img/project_CIN_inference_theory_1.jpg" class="img-fluid rounded z-depth-1" width="100%" height="auto" title="example image" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> </div> <div class="caption"> Top: The inference method relies on ABC-rf to find parameter posterior distributions. Bottom: Overview of some statistics based on single-cell phylogeny (left) and distance between observed and simulated CN profiles (right). </div> <p>Using this framework, we are able to uncover the true values of both the missegregation probability and the selection parameters for individual chromosomes. This comparison was performed with synthetic tests, where we know the true values against which the parameter posterior distributions can be compared. The posterior distributions are unimodal, which indicates identifiability, and center around the ground truth values.</p> <div class="row"> <div class="col-sm mt-3 mt-md-0"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/project_CIN_inference_theory_2-480.webp 480w,/assets/img/project_CIN_inference_theory_2-800.webp 800w,/assets/img/project_CIN_inference_theory_2-1400.webp 1400w," type="image/webp" sizes="95vw"></source> <img src="/assets/img/project_CIN_inference_theory_2.jpg" class="img-fluid rounded z-depth-1" width="100%" height="auto" title="example image" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> </div> <div class="caption"> Inference of missegregation probability (top left) and selection parameters for individual chromosomes in synthetic testing. For each parameter, the posterior distribution (dark blue; broken lines indicate mean, median and mode) is inferred from a uniform prior distribution (light blue), compared to the ground truth value (black line). The inference is accurate if the posterior distribution centers around ground truth value. </div> <p>Importantly, the accuracy and uncertainty of the results (quantified as Root mean square error (RMSE) and standard deviation of the posteriors, respectively) do not increase significantly for small sample sizes of either single-cell or bulk cohorts. Minimum error can be achieved with as few as 40 sc and 50 bulk samples, while minimum uncertainty is reached with 20 and 30 bulk samples. These requirements are already satisfied with currently available data for some cancer types. We expect that as DNA sequencing becomes an essential part of cancer diagnosis, our simulation and inference framework will become a pivotal tool to analyze the data toward better understanding of cancer evolution and adaptation.</p> <div class="row"> <div class="col-sm mt-3 mt-md-0"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/project_CIN_inference_theory_3-480.webp 480w,/assets/img/project_CIN_inference_theory_3-800.webp 800w,/assets/img/project_CIN_inference_theory_3-1400.webp 1400w," type="image/webp" sizes="95vw"></source> <img src="/assets/img/project_CIN_inference_theory_3.jpg" class="img-fluid rounded z-depth-1" width="100%" height="auto" title="example image" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> </div> <div class="caption"> RMSE and standard deviation of the posterior distributions, as a function of sample size of the single-cell cohort (left) and the bulk data (right) </div> <hr> <p>ABC-rf’s insensitivity to noise is essential for incorporating a large number of statistics in our inference method. It therefore offers an incredible advantage over traditional ABC methods, which require selecting substantially important hyperparameters and therefore are not flexible for such expansive problems and models such as ours. However, ABC-rf often requires a large training set and therefore long running time. To improve its performance, we developed Approximate Bayesian Computation sequential Monte Carlo with (distributional) random forests (ABC-SMC-RF) <a class="citation" href="#dinh2024approximate">(Dinh et al., 2024)</a>.</p> <p>A tree in the random forest grows from a root node, composed of a subsample or bootstrap sample of the training set. The algorithm then repeatedly divides each node, each time by choosing a particular statistic and segregating the node’s simulations into those whose values are lower or higher than a threshold. The choice of statistic and threshold at each node is made such that the simulations in each child node are most “similar” to each other, with respect to the given statistic. Finally, the algorithm traverses each tree with the statistics measured from the data. Simulations in the same leaf that the data ends up in are deemed closer to the data, hence the parameters of those simulations form an approximation for the posterior distribution.</p> <p>ABC-SMC-RF incorporates the random forest within the framework of sequential Monte Carlo (SMC), and it is available as an <a href="https://github.com/dinhngockhanh/abcsmcrf" rel="external nofollow noopener" target="_blank">R package on Github</a>. In SMC, the posterior distribution is improved through successive iterations from the prior distribution. Each iteration in ABC-SMC-RF samples the parameters from the previous iteration’s posterior, perturbs the parameters to maintain parameter diversity, then infers the next posterior distribution with random forest.</p> <div class="row"> <div class="col-sm mt-3 mt-md-0"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/project_CIN_inference_abcsmcdrf_1-480.webp 480w,/assets/img/project_CIN_inference_abcsmcdrf_1-800.webp 800w,/assets/img/project_CIN_inference_abcsmcdrf_1-1400.webp 1400w," type="image/webp" sizes="95vw"></source> <img src="/assets/img/project_CIN_inference_abcsmcdrf_1.jpg" class="img-fluid rounded z-depth-1" width="100%" height="auto" title="example image" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> </div> <div class="caption"> Left: Schematic for the combination of ABC-SMC-RF and CINner toward inferring parameter posterior distributions. Middle: Schematic for constructing a tree in the random forest. Right: Schematic for inferring a parameter estimate from the tree. </div> <p>In test studies, ABC-SMC-RF performs well across a wide range of mathematical models, including both deterministic and stochastic models with varying complexity levels and parameter counts. Its results are on par with traditional ABC methods with carefully calibrated hyperparameters. Moreover, ABC-SMC-RF tends to perfom better than previous random forest methods, which are also calibration-free. This is because in those algorithms, most simulations with parameters sampled directly from the prior distribution are significantly different from the data. They therefore require a large training set to select the most relevant parameter regions. By comparison, ABC-SMC-RF iteratively updates the parameter distributions, therefore more of the simulations are relevant to the data. Another practical advantage is that the random forest in each iteration can be trained on smaller training sets, therefore reducing computational runtime and storage requirement.</p> <div class="row"> <div class="col-sm mt-3 mt-md-0"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/project_CIN_inference_abcsmcdrf_2-480.webp 480w,/assets/img/project_CIN_inference_abcsmcdrf_2-800.webp 800w,/assets/img/project_CIN_inference_abcsmcdrf_2-1400.webp 1400w," type="image/webp" sizes="95vw"></source> <img src="/assets/img/project_CIN_inference_abcsmcdrf_2.jpg" class="img-fluid rounded z-depth-1" width="100%" height="auto" title="example image" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> </div> <div class="caption"> Comparison of posterior distributions from ABC-SMC-RF (red) with a previous ABC-RF method (blue) and ABC-SMC with calibrated hyperparameters (yellow) for the Lotka-Volterra model. </div> <hr> </article> <h2>References</h2> <div class="publications"> <h2 class="bibliography">2025</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/dinh2024cinner-480.webp 480w,/assets/img/publication_preview/dinh2024cinner-800.webp 800w,/assets/img/publication_preview/dinh2024cinner-1400.webp 1400w," type="image/webp" sizes="200px"> <img src="/assets/img/publication_preview/dinh2024cinner.jpg" class="preview z-depth-1 rounded" width="100%" height="auto" alt="dinh2024cinner.jpg" data-zoomable loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </source></picture> </figure> </div> <div id="dinh2024cinner" class="col-sm-8"> <div class="title">CINner: modeling and simulation of chromosomal instability in cancer at single-cell resolution</div> <div class="author"> <em>Khanh N. Dinh</em>, Ignacio Vázquez-Garcı́a, Andrew Chan, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Rhea Malhotra, Adam Weiner, Andrew W. McPherson, Simon Tavaré' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>PLoS Computational Biology</em>, 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1371/journal.pcbi.1012902" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1012902" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/dinh2024cinner.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1371/journal.pcbi.1012902" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Cancer development is characterized by chromosomal instability, manifesting in frequent occurrences of different genomic alteration mechanisms ranging in extent and impact. Mathematical modeling can help evaluate the role of each mutational process during tumor progression, however existing frameworks can only capture certain aspects of chromosomal instability (CIN). We present CINner, a mathematical framework for modeling genomic diversity and selection during tumor evolution. The main advantage of CINner is its flexibility to incorporate many genomic events that directly impact cellular fitness, from driver gene mutations to copy number alterations (CNAs), including focal amplifications and deletions, missegregations and whole-genome duplication (WGD). We apply CINner to find chromosome-arm selection parameters that drive tumorigenesis in the absence of WGD in chromosomally unstable cancer types from the Pan-Cancer Analysis of Whole Genomes (PCAWG, n=718). We found that the selection parameters predict WGD prevalence among different chromosomally unstable tumors, hinting that the selective advantage of WGD cells hinges on their tolerance for aneuploidy and escape from nullisomy. Analysis of inference results using CINner across cancer types in The Cancer Genome Atlas (n=8207) further reveals that the inferred selection parameters reflect the bias between tumor suppressor genes and oncogenes on specific genomic regions. Direct application of CINner to model the WGD proportion and fraction of genome altered (FGA) in PCAWG uncovers the increase in CNA probabilities associated with WGD in each cancer type. CINner can also be utilized to study chromosomally stable cancer types, by applying a selection model based on driver gene mutations and focal amplifications or deletions (chronic lymphocytic leukemia in PCAWG, n=95). Finally, we used CINner to analyze the impact of CNA probabilities, chromosome selection parameters, tumor growth dynamics and population size on cancer fitness and heterogeneity. We expect that CINner will provide a powerful modeling tool for the oncology community to quantify the impact of newly uncovered genomic alteration mechanisms on shaping tumor progression and adaptation.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">dinh2024cinner</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{CINner: modeling and simulation of chromosomal instability in cancer at single-cell resolution}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Dinh, Khanh N. and V{\'a}zquez-Garc{\'\i}a, Ignacio and Chan, Andrew and Malhotra, Rhea and Weiner, Adam and McPherson, Andrew W. and Tavar{\'e}, Simon}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{PLoS Computational Biology}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2025}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1371/journal.pcbi.1012902}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/mcpherson2025ongoing-480.webp 480w,/assets/img/publication_preview/mcpherson2025ongoing-800.webp 800w,/assets/img/publication_preview/mcpherson2025ongoing-1400.webp 1400w," type="image/webp" sizes="200px"> <img src="/assets/img/publication_preview/mcpherson2025ongoing.jpg" class="preview z-depth-1 rounded" width="100%" height="auto" alt="mcpherson2025ongoing.jpg" data-zoomable loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </source></picture> </figure> </div> <div id="mcpherson2025ongoing" class="col-sm-8"> <div class="title">Ongoing genome doubling shapes evolvability and immunity in ovarian cancer</div> <div class="author"> <a href="https://www.mskcc.org/profile/andrew-mcpherson" rel="external nofollow noopener" target="_blank">Andrew W. McPherson</a>, Ignacio Vázquez-Garcı́a, Matthew A. Myers, and <span class="more-authors" title="click to view 55 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '55 more authors' ? 'Duaa H. Al-Rawi, Matthew Zatzman, Adam C. Weiner, Samuel S. Freeman, Neeman Mohibullah, Gryte Satas, Marc J. Williams, Nicholas Ceglia, Danguolė Norbūnaitė, Allen W. Zhang, Jun Li, Jamie L. P. Lim, Michele Wu, Seongmin Choi, Eliyahu Havasov, Diljot Grewal, Hongyu Shi, Minsoo Kim, Roland F. Schwarz, Tom Kaufmann, Khanh N. Dinh, Florian Uhlitz, Julie Tran, Yushi Wu, Ruchi Patel, Satish Ramakrishnan, DooA Kim, Justin Clarke, Hunter Green, Emily Ali, Melody DiBona, Nancy Varice, Ritika Kundra, Vance Broach, Ginger J. Gardner, Kara L. Roche, Yukio Sonoda, Oliver Zivanovic, Sarah H. Kim, Rachel N. Grisham, Ying L. Liu, Agnes Viale, Nicole Rusk, Yulia Lakhman, Lora H. Ellenson, Simon Tavaré, Samuel Aparicio, Dennis S. Chi, Carol Aghajanian, Nadeem R. Abu-Rustum, Claire F. Friedman, Dmitriy Zamarin, Britta Weigelt, Samuel F. Bakhoum, Sohrab P. Shah' : '55 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">55 more authors</span> </div> <div class="periodical"> <em>Nature</em>, 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1038/s41586-025-09240-3" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.nature.com/articles/s41586-025-09240-3" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41586-025-09240-3" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Whole-genome doubling (WGD) is a common feature of human cancers and is linked to tumour progression, drug resistance, and metastasis. Here we examine the impact of WGD on somatic evolution and immune evasion at single-cell resolution in patient tumours. Using single-cell whole-genome sequencing, we analysed 70 high-grade serous ovarian cancer samples from 41 patients (30,260 tumour genomes) and observed near-ubiquitous evidence that WGD is an ongoing mutational process. WGD was associated with increased cell-cell diversity and higher rates of chromosomal missegregation and consequent micronucleation. We developed a mutation-based WGD timing method called doubleTime to delineate specific modes by which WGD can drive tumour evolution, including early fixation followed by considerable diversification, multiple parallel WGD events on a pre-existing background of copy-number diversity, and evolutionarily late WGD in small clones and individual cells. Furthermore, using matched single-cell RNA sequencing and high-resolution immunofluorescence microscopy, we found that inflammatory signalling and cGAS-STING pathway activation result from ongoing chromosomal instability, but this is restricted to predominantly diploid tumours (WGD-low). By contrast, predominantly WGD tumours (WGD-high), despite increased missegregation, exhibited cell-cycle dysregulation, STING1 repression, and immunosuppressive phenotypic states. Together, these findings establish WGD as an ongoing mutational process that promotes evolvability and dysregulated immunity in high-grade serous ovarian cancer.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">mcpherson2025ongoing</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Ongoing genome doubling shapes evolvability and immunity in ovarian cancer}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{McPherson, Andrew W. and V{\'a}zquez-Garc{\'\i}a, Ignacio and Myers, Matthew A. and Al-Rawi, Duaa H. and Zatzman, Matthew and Weiner, Adam C. and Freeman, Samuel S. and Mohibullah, Neeman and Satas, Gryte and Williams, Marc J. and Ceglia, Nicholas and Norb\={u}nait\.{e}, Danguol\.{e} and Zhang, Allen W. and Li, Jun and Lim, Jamie L. P. and Wu, Michele and Choi, Seongmin and Havasov, Eliyahu and Grewal, Diljot and Shi, Hongyu and Kim, Minsoo and Schwarz, Roland F. and Kaufmann, Tom and Dinh, Khanh N. and Uhlitz, Florian and Tran, Julie and Wu, Yushi and Patel, Ruchi and Ramakrishnan, Satish and Kim, DooA and Clarke, Justin and Green, Hunter and Ali, Emily and DiBona, Melody and Varice, Nancy and Kundra, Ritika and Broach, Vance and Gardner, Ginger J. and Roche, Kara L. and Sonoda, Yukio and Zivanovic, Oliver and Kim, Sarah H. and Grisham, Rachel N. and Liu, Ying L. and Viale, Agnes and Rusk, Nicole and Lakhman, Yulia and Ellenson, Lora H. and Tavar{\'e}, Simon and Aparicio, Samuel and Chi, Dennis S. and Aghajanian, Carol and Abu-Rustum, Nadeem R. and Friedman, Claire F. and Zamarin, Dmitriy and Weigelt, Britta and Bakhoum, Samuel F. and Shah, Sohrab P.}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Nature}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2025}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Nature Publishing Group UK London}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/s41586-025-09240-3}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> </ol> <h2 class="bibliography">2024</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/xiang2024inference-480.webp 480w,/assets/img/publication_preview/xiang2024inference-800.webp 800w,/assets/img/publication_preview/xiang2024inference-1400.webp 1400w," type="image/webp" sizes="200px"> <img src="/assets/img/publication_preview/xiang2024inference.jpg" class="preview z-depth-1 rounded" width="100%" height="auto" alt="xiang2024inference.jpg" data-zoomable loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </source></picture> </figure> </div> <div id="xiang2024inference" class="col-sm-8"> <div class="title">Inference of chromosome selection parameters and missegregation rate in cancer from DNA-sequencing data</div> <div class="author"> Zijin Xiang, Zhihan Liu, and <em>Khanh N. Dinh</em> </div> <div class="periodical"> <em>Scientific Reports</em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1038/s41598" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.nature.com/articles/s41598-024-67842-9" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/xiang2024inference.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41598" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Aneuploidy is frequently observed in cancers and has been linked to poor patient outcome. Analysis of aneuploidy in DNA-sequencing (DNA-seq) data necessitates untangling the effects of the Copy Number Aberration (CNA) occurrence rates and the selection coefficients that act upon the resulting karyotypes. We introduce a parameter inference algorithm that takes advantage of both bulk and single-cell DNA-seq cohorts. The method is based on Approximate Bayesian Computation (ABC) and utilizes CINner, our recently introduced simulation algorithm of chromosomal instability in cancer. We examine three groups of statistics to summarize the data in the ABC routine: (A) Copy Number-based measures, (B) phylogeny tip statistics, and (C) phylogeny balance indices. Using these statistics, our method can recover both the CNA probabilities and selection parameters from ground truth data, and performs well even for data cohorts of relatively small sizes. We find that only statistics in groups A and C are well-suited for identifying CNA probabilities, and only group A carries the signals for estimating selection parameters. Moreover, the low number of CNA events at large scale compared to cell counts in single-cell samples means that statistics in group B cannot be estimated accurately using phylogeny reconstruction algorithms at the chromosome level. As data from both bulk and single-cell DNA-sequencing techniques becomes increasingly available, our inference framework promises to facilitate the analysis of distinct cancer types, differentiation between selection and neutral drift, and prediction of cancer clonal dynamics.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">xiang2024inference</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Inference of chromosome selection parameters and missegregation rate in cancer from DNA-sequencing data}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Xiang, Zijin and Liu, Zhihan and Dinh, Khanh N.}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Scientific Reports}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{14}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{17699}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Nature Publishing Group UK London}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/s41598}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/dinh2024approximate-480.webp 480w,/assets/img/publication_preview/dinh2024approximate-800.webp 800w,/assets/img/publication_preview/dinh2024approximate-1400.webp 1400w," type="image/webp" sizes="200px"> <img src="/assets/img/publication_preview/dinh2024approximate.jpg" class="preview z-depth-1 rounded" width="100%" height="auto" alt="dinh2024approximate.jpg" data-zoomable loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </source></picture> </figure> </div> <div id="dinh2024approximate" class="col-sm-8"> <div class="title">Approximate Bayesian Computation sequential Monte Carlo via random forests</div> <div class="author"> <em>Khanh N. Dinh</em>, Zijin Xiang, Zhihan Liu, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Simon Tavaré' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>arXiv</em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.48550/arXiv.2406.15865" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.arxiv.org/abs/2406.15865" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/dinh2024approximate.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.48550/arXiv.2406.15865" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Approximate Bayesian Computation (ABC) is a popular inference method when likelihoods are hard to come by. Practical bottlenecks of ABC applications include selecting statistics that summarize the data without losing too much information or introducing uncertainty, and choosing distance functions and tolerance thresholds that balance accuracy and computational efficiency. Recent studies have shown that ABC methods using random forest (RF) methodology perform well while circumventing many of ABC’s drawbacks. However, RF construction is computationally expensive for large numbers of trees and model simulations, and there can be high uncertainty in the posterior if the prior distribution is uninformative. Here we adapt distributional random forests to the ABC setting, and introduce Approximate Bayesian Computation sequential Monte Carlo with random forests (ABC-SMC-(D)RF). This updates the prior distribution iteratively to focus on the most likely regions in the parameter space. We show that ABC-SMC-(D)RF can accurately infer posterior distributions for a wide range of deterministic and stochastic models in different scientific areas.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">dinh2024approximate</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Approximate Bayesian Computation sequential Monte Carlo via random forests}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Dinh, Khanh N. and Xiang, Zijin and Liu, Zhihan and Tavar{\'e}, Simon}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{arXiv}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.48550/arXiv.2406.15865}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> </ol> <h2 class="bibliography">2022</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/funnell2022single-480.webp 480w,/assets/img/publication_preview/funnell2022single-800.webp 800w,/assets/img/publication_preview/funnell2022single-1400.webp 1400w," type="image/webp" sizes="200px"> <img src="/assets/img/publication_preview/funnell2022single.jpg" class="preview z-depth-1 rounded" width="100%" height="auto" alt="funnell2022single.jpg" data-zoomable loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </source></picture> </figure> </div> <div id="funnell2022single" class="col-sm-8"> <div class="title">Single-cell genomic variation induced by mutational processes in cancer</div> <div class="author"> Tyler Funnell, Ciara H O’Flanagan, Marc J Williams, and <span class="more-authors" title="click to view 51 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '51 more authors' ? 'Andrew McPherson, Steven McKinney, Farhia Kabeer, Hakwoo Lee, Sohrab Salehi, Ignacio Vázquez-Garcı́a, Hongyu Shi, Emily Leventhal, Tehmina Masud, Peter Eirew, Damian Yap, Allen W. Zhang, Jamie L. P. Lim, Beixi Wang, Jazmine Brimhall, Justina Biele, Jerome Ting, Vinci Au, Michael Van Vliet, Yi Fei Liu, Sean Beatty, Daniel Lai, Jenifer Pham, Diljot Grewal, Douglas Abrams, Eliyahu Havasov, Samantha Leung, Viktoria Bojilova, Richard A. Moore, Nicole Rusk, Florian Uhlitz, Nicholas Ceglia, Adam C. Weiner, Elena Zaikova, J. Maxwell Douglas, Dmitriy Zamarin, Britta Weigelt, Sarah H. Kim, Arnaud Da Cruz Paula, Jorge S. Reis-Filho, Spencer D. Martin, Yangguang Li, Hong Xu, Teresa Ruiz Algara, So Ra Lee, Viviana Cerda Llanos, David G. Huntsman, Jessica N. McAlpine, IMAXT Consortium, Sohrab P. Shah, Samuel Aparicio' : '51 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">51 more authors</span> <i class="fa-solid fa-circle-info ml-1" data-toggle="popover" data-placement="top" data-html="true" data-content="As part of the CRUK IMAXT Grand Challenge Team"> </i> </div> <div class="periodical"> <em>Nature</em>, 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1038/s41586-022-05249-0" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.nature.com/articles/s41586-022-05249-0" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/funnell2022single.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41586-022-05249-0" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>How cell-to-cell copy number alterations that underpin genomic instability in human cancers drive genomic and phenotypic variation, and consequently the evolution of cancer, remains understudied. Here, by applying scaled single-cell whole-genome sequencing to wild-type, TP53-deficient and TP53-deficient; BRCA1-deficient or TP53-deficient; BRCA2-deficient mammary epithelial cells (13,818 genomes), and to primary triple-negative breast cancer (TNBC) and high-grade serous ovarian cancer (HGSC) cells (22,057 genomes), we identify three distinct ’foreground’ mutational patterns that are defined by cell-to-cell structural variation. Cell- and clone-specific high-level amplifications, parallel haplotype-specific copy number alterations and copy number segment length variation (serrate structural variations) had measurable phenotypic and evolutionary consequences. In TNBC and HGSC, clone-specific high-level amplifications in known oncogenes were highly prevalent in tumours bearing fold-back inversions, relative to tumours with homologous recombination deficiency, and were associated with increased clone-to-clone phenotypic variation. Parallel haplotype-specific alterations were also commonly observed, leading to phylogenetic evolutionary diversity and clone-specific mono-allelic expression. Serrate variants were increased in tumours with fold-back inversions and were highly correlated with increased genomic diversity of cellular populations. Together, our findings show that cell-to-cell structural variation contributes to the origins of phenotypic and evolutionary diversity in TNBC and HGSC, and provide insight into the genomic and mutational states of individual cancer cells.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">funnell2022single</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Single-cell genomic variation induced by mutational processes in cancer}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Funnell, Tyler and O'Flanagan, Ciara H and Williams, Marc J and McPherson, Andrew and McKinney, Steven and Kabeer, Farhia and Lee, Hakwoo and Salehi, Sohrab and V{\'a}zquez-Garc{\'\i}a, Ignacio and Shi, Hongyu and Leventhal, Emily and Masud, Tehmina and Eirew, Peter and Yap, Damian and Zhang, Allen W. and Lim, Jamie L. P. and Wang, Beixi and Brimhall, Jazmine and Biele, Justina and Ting, Jerome and Au, Vinci and Van Vliet, Michael and Liu, Yi Fei and Beatty, Sean and Lai, Daniel and Pham, Jenifer and Grewal, Diljot and Abrams, Douglas and Havasov, Eliyahu and Leung, Samantha and Bojilova, Viktoria and Moore, Richard A. and Rusk, Nicole and Uhlitz, Florian and Ceglia, Nicholas and Weiner, Adam C. and Zaikova, Elena and Douglas, J. Maxwell and Zamarin, Dmitriy and Weigelt, Britta and Kim, Sarah H. and Da Cruz Paula, Arnaud and Reis-Filho, Jorge S. and Martin, Spencer D. and Li, Yangguang and Xu, Hong and de Algara, Teresa Ruiz and Lee, So Ra and Llanos, Viviana Cerda and Huntsman, David G. and McAlpine, Jessica N. and Consortium, IMAXT and Shah, Sohrab P. and Aparicio, Samuel}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Nature}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{612}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{7938}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{106--115}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2022}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Nature Publishing Group UK London}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/s41586-022-05249-0}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li></ol> <h2 class="bibliography">2021</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/salehi2021clonal-480.webp 480w,/assets/img/publication_preview/salehi2021clonal-800.webp 800w,/assets/img/publication_preview/salehi2021clonal-1400.webp 1400w," type="image/webp" sizes="200px"> <img src="/assets/img/publication_preview/salehi2021clonal.jpg" class="preview z-depth-1 rounded" width="100%" height="auto" alt="salehi2021clonal.jpg" data-zoomable loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </source></picture> </figure> </div> <div id="salehi2021clonal" class="col-sm-8"> <div class="title">Clonal fitness inferred from time-series modelling of single-cell cancer genomes</div> <div class="author"> Sohrab Salehi, Farhia Kabeer, Nicholas Ceglia, and <span class="more-authors" title="click to view 31 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '31 more authors' ? 'Mirela Andronescu, Marc J Williams, Kieran R Campbell, Tehmina Masud, Beixi Wang, Justina Biele, Jazmine Brimhall, David Gee, Hakwoo Lee, Jerome Ting, Allen W. Zhang, Hoa Tran, Ciara O’Flanagan, Fatemeh Dorri, Nicole Rusk, Teresa Ruiz Algara, So Ra Lee, Brian Yu Chieh Cheng, Peter Eirew, Takako Kono, Jenifer Pham, Diljot Grewal, Daniel Lai, Richard Moore, Andrew J. Mungall, Marco A. Marra, IMAXT Consortium, Andrew McPherson, Alexandre Bouchard-Côté, Samuel Aparicio, Sohrab P. Shah' : '31 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">31 more authors</span> <i class="fa-solid fa-circle-info ml-1" data-toggle="popover" data-placement="top" data-html="true" data-content="As part of the CRUK IMAXT Grand Challenge Team"> </i> </div> <div class="periodical"> <em>Nature</em>, 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1038/s41586-021-03648-3" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.nature.com/articles/s41586-021-03648-3" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/salehi2021clonal.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41586-021-03648-3" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Progress in defining genomic fitness landscapes in cancer, especially those defined by copy number alterations (CNAs), has been impeded by lack of time-series single-cell sampling of polyclonal populations and temporal statistical models. Here we generated 42,000 genomes from multi-year time-series single-cell whole-genome sequencing of breast epithelium and primary triple-negative breast cancer (TNBC) patient-derived xenografts (PDXs), revealing the nature of CNA-defined clonal fitness dynamics induced by TP53 mutation and cisplatin chemotherapy. Using a new Wright-Fisher population genetics model to infer clonal fitness, we found that TP53 mutation alters the fitness landscape, reproducibly distributing fitness over a larger number of clones associated with distinct CNAs. Furthermore, in TNBC PDX models with mutated TP53, inferred fitness coefficients from CNA-based genotypes accurately forecast experimentally enforced clonal competition dynamics. Drug treatment in three long-term serially passaged TNBC PDXs resulted in cisplatin-resistant clones emerging from low-fitness phylogenetic lineages in the untreated setting. Conversely, high-fitness clones from treatment-naive controls were eradicated, signalling an inversion of the fitness landscape. Finally, upon release of drug, selection pressure dynamics were reversed, indicating a fitness cost of treatment resistance. Together, our findings define clonal fitness linked to both CNA and therapeutic resistance in polyclonal tumours.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">salehi2021clonal</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Clonal fitness inferred from time-series modelling of single-cell cancer genomes}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Salehi, Sohrab and Kabeer, Farhia and Ceglia, Nicholas and Andronescu, Mirela and Williams, Marc J and Campbell, Kieran R and Masud, Tehmina and Wang, Beixi and Biele, Justina and Brimhall, Jazmine and Gee, David and Lee, Hakwoo and Ting, Jerome and Zhang, Allen W. and Tran, Hoa and O’Flanagan, Ciara and Dorri, Fatemeh and Rusk, Nicole and de Algara, Teresa Ruiz and Lee, So Ra and Cheng, Brian Yu Chieh and Eirew, Peter and Kono, Takako and Pham, Jenifer and Grewal, Diljot and Lai, Daniel and Moore, Richard and Mungall, Andrew J. and Marra, Marco A. and Consortium, IMAXT and McPherson, Andrew and Bouchard-C{\o}t{\'e}, Alexandre and Aparicio, Samuel and Shah, Sohrab P.}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Nature}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{595}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{7868}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{585--590}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Nature Publishing Group UK London}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/s41586-021-03648-3}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li></ol> <h2 class="bibliography">2019</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/laks2019clonal-480.webp 480w,/assets/img/publication_preview/laks2019clonal-800.webp 800w,/assets/img/publication_preview/laks2019clonal-1400.webp 1400w," type="image/webp" sizes="200px"> <img src="/assets/img/publication_preview/laks2019clonal.jpg" class="preview z-depth-1 rounded" width="100%" height="auto" alt="laks2019clonal.jpg" data-zoomable loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </source></picture> </figure> </div> <div id="laks2019clonal" class="col-sm-8"> <div class="title">Clonal decomposition and DNA replication states defined by scaled single-cell genome sequencing</div> <div class="author"> Emma Laks, <a href="https://www.mskcc.org/profile/andrew-mcpherson" rel="external nofollow noopener" target="_blank">Andrew McPherson</a>, Hans Zahn, and <span class="more-authors" title="click to view 48 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '48 more authors' ? 'Daniel Lai, Adi Steif, Jazmine Brimhall, Justina Biele, Beixi Wang, Tehmina Masud, Jerome Ting, Diljot Grewal, Cydney Nielsen, Samantha Leung, Viktoria Bojilova, Maia Smith, Oleg Golovko, Steven Poon, Peter Eirew, Farhia Kabeer, Teresa Algara, So Ra Lee, M. Jafar Taghiyar, Curtis Huebner, Jessica Ngo, Tim Chan, Spencer Vatrt-Watts, Pascale Walters, Nafis Abrar, Sophia Chan, Matt Wiens, Lauren Martin, R. Wilder Scott, T. Michael Underhill, Elizabeth Chavez, Christian Steidl, Daniel Da Costa, Yussanne Ma, Robin J.N. Coope, Richard Corbett, Stephen Pleasance, Richard Moore, Andrew J. Mungall, Colin Mar, Fergus Cafferty, Karen Gelmon, Stephen Chia, IMAXT Consortium, Marco A. Marra, Carl Hansen, Sohrab P. Shah, Samuel Aparicio' : '48 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">48 more authors</span> <i class="fa-solid fa-circle-info ml-1" data-toggle="popover" data-placement="top" data-html="true" data-content="As part of the CRUK IMAXT Grand Challenge Team"> </i> </div> <div class="periodical"> <em>Cell</em>, 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1016/j.cell.2019.10.026" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.cell.com/cell/fulltext/S0092-8674(19)31176-6" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/laks2019clonal.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.cell.2019.10.026" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Accurate measurement of clonal genotypes, mutational processes, and replication states from individual tumor-cell genomes will facilitate improved understanding of tumor evolution. We have developed DLP+, a scalable single-cell whole-genome sequencing platform implemented using commodity instruments, image-based object recognition, and open source computational methods. Using DLP+, we have generated a resource of 51,926 single-cell genomes and matched cell images from diverse cell types including cell lines, xenografts, and diagnostic samples with limited material. From this resource we have defined variation in mitotic mis-segregation rates across tissue types and genotypes. Analysis of matched genomic and image measurements revealed correlations between cellular morphology and genome ploidy states. Aggregation of cells sharing copy number profiles allowed for calculation of single-nucleotide resolution clonal genotypes and inference of clonal phylogenies and avoided the limitations of bulk deconvolution. Finally, joint analysis over the above features defined clone-specific chromosomal aneuploidy in polyclonal populations.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">laks2019clonal</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Clonal decomposition and DNA replication states defined by scaled single-cell genome sequencing}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Laks, Emma and McPherson, Andrew and Zahn, Hans and Lai, Daniel and Steif, Adi and Brimhall, Jazmine and Biele, Justina and Wang, Beixi and Masud, Tehmina and Ting, Jerome and Grewal, Diljot and Nielsen, Cydney and Leung, Samantha and Bojilova, Viktoria and Smith, Maia and Golovko, Oleg and Poon, Steven and Eirew, Peter and Kabeer, Farhia and Ruiz de Algara, Teresa and Lee, So Ra and Taghiyar, M. Jafar and Huebner, Curtis and Ngo, Jessica and Chan, Tim and Vatrt-Watts, Spencer and Walters, Pascale and Abrar, Nafis and Chan, Sophia and Wiens, Matt and Martin, Lauren and Scott, R. Wilder and Underhill, T. Michael and Chavez, Elizabeth and Steidl, Christian and Da Costa, Daniel and Ma, Yussanne and Coope, Robin J.N. and Corbett, Richard and Pleasance, Stephen and Moore, Richard and Mungall, Andrew J. and Mar, Colin and Cafferty, Fergus and Gelmon, Karen and Chia, Stephen and Consortium, IMAXT and Marra, Marco A. and Hansen, Carl and Shah, Sohrab P. and Aparicio, Samuel}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Cell}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{179}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{5}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1207--1221}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2019}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Elsevier}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.cell.2019.10.026}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li></ol> </div> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2025 Khanh N. Dinh. Last updated: September 03, 2025. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2781658a0a2b13ed609542042a859126"></script> <script defer src="/assets/js/common.js?e0514a05c5c95ac1a93a8dfd5249b92e"></script> <script defer src="/assets/js/copy_code.js?12775fdf7f95e901d7119054556e495f" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.2/es5/tex-mml-chtml.js" integrity="sha256-MASABpB4tYktI2Oitl4t+78w/lyA+D7b/s9GEP0JOGI=" crossorigin="anonymous"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6" crossorigin="anonymous"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> <script src="/assets/js/vanilla-back-to-top.min.js?f40d453793ff4f64e238e420181a1d17"></script> <script>addBackToTop();</script> </body> </html>