<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> publications | Khanh N. Dinh Lab </title> <meta name="author" content="Khanh N. Dinh"> <meta name="description" content="Website of Dr. Khanh N. Dinh's Lab."> <meta name="keywords" content="Computational-biology, Mathematical-modeling"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%F0%9F%A7%AC&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://dinhngockhanh.github.io/publications/"> <script src="/assets/js/theme.js?9a0c749ec5240d9cda97bc72359a72c0"></script> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script>initTheme();</script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> Khanh N. Dinh Lab </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about </a> </li> <li class="nav-item "> <a class="nav-link" href="/research/">research </a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">publications <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/repositories/">repositories </a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">cv </a> </li> <li class="nav-item "> <a class="nav-link" href="/alumni/">alumni </a> </li> <li class="nav-item "> <a class="nav-link" href="/people/">people </a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="ti ti-sun-moon" id="light-toggle-system"></i> <i class="ti ti-moon-filled" id="light-toggle-dark"></i> <i class="ti ti-sun-filled" id="light-toggle-light"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">publications</h1> <p class="post-description"></p> </header> <article> <div class="publications"> <h2 class="bibliography">2026</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/wang2026bestish-480.webp 480w,/assets/img/publication_preview/wang2026bestish-800.webp 800w,/assets/img/publication_preview/wang2026bestish-1400.webp 1400w," type="image/webp" sizes="200px"></source> <img src="/assets/img/publication_preview/wang2026bestish.jpg" class="preview z-depth-1 rounded" width="100%" height="auto" alt="wang2026bestish.jpg" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="wang2026bestish" class="col-sm-8"> <div class="title">BESTish: A diffusion-approximation framework for inferring selection and mutation in clonal hematopoiesis</div> <div class="author"> Ren-Yi Wang<sup>*</sup>, <em>Khanh N. Dinh<sup>*</sup></em>, Keito Taketomi, Guodong Pang, Katherine Y. King, and <a href="https://profiles.rice.edu/faculty/marek-kimmel" rel="external nofollow noopener" target="_blank">Marek Kimmel</a> </div> <div class="periodical"> <em>bioRxiv</em><ul> <li>denotes co-first authors</li> </ul> , 2026 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.64898/2026.01.27.702030" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.biorxiv.org/content/10.64898/2026.01.27.702030v1" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/wang2026bestish.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.64898/2026.01.27.702030" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Clonal hematopoiesis (CH) arises when hematopoietic stem cells (HSCs) gain a fitness advantage from somatic mutations and expand, resulting in an increase in variant allele frequency (VAF) over time. To analyze CH trajectories, we develop a state-dependent stochastic model of wild-type and mutant HSCs, in which an environmental parameter α∈[0,1] regulates death rates and interpolates between homeostatic (Moran-like, α=1) and growth-facilitating (α&lt;1) regimes. Using functional law of large numbers and central limit theorems, we derive explicit mean-field dynamics and a Gaussian-Markov approximation for VAF fluctuations. We show that the mean VAF trajectory has an explicit logistic form determined by selective advantage, while environmental effects affect only the variance and autocovariance structure. Building on these results, we introduce BESTish (Bayesian estimate for selection incorporating scaling-limit to detect mutant heterogeneity), a novel, efficient and accurate Bayesian inference method that can be applied to analyze both cohort-level and longitudinal VAF datasets. BESTish implements the closed-form finite-dimensional distributions that we derive to estimate mutation fitness, mutation rate, and environmental strength for individual CH drivers. When applied to existing CH datasets, BESTish produces consistent mutation fitness inferences across different studies, and estimates CH driver mutation rates in agreement with independent experimental studies. Furthermore, BESTish reveals patient-specific heterogeneity in the selective behavior of recurrent mutations, and identifies variants whose dynamics are compatible with non-homeostatic, growth-facilitating environments. BESTish provides a unified and mechanistic framework for quantifying CH evolution, with potential applications for other biological systems where clonal expansions can be measured.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">wang2026bestish</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{BESTish: A diffusion-approximation framework for inferring selection and mutation in clonal hematopoiesis}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Wang, Ren-Yi and Dinh, Khanh N. and Taketomi, Keito and Pang, Guodong and King, Katherine Y. and Kimmel, Marek}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{bioRxiv}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2026}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.64898/2026.01.27.702030}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li></ol> <h2 class="bibliography">2025</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/dinh2025approximate-480.webp 480w,/assets/img/publication_preview/dinh2025approximate-800.webp 800w,/assets/img/publication_preview/dinh2025approximate-1400.webp 1400w," type="image/webp" sizes="200px"></source> <img src="/assets/img/publication_preview/dinh2025approximate.jpg" class="preview z-depth-1 rounded" width="100%" height="auto" alt="dinh2025approximate.jpg" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="dinh2025approximate" class="col-sm-8"> <div class="title">Approximate Bayesian computation sequential Monte Carlo via random forests</div> <div class="author"> <em>Khanh N. Dinh</em>, Cécile Liu, Zijin Xiang, Zhihan Liu, and <a href="https://tavarelab.cancerdynamics.columbia.edu" rel="external nofollow noopener" target="_blank">Simon Tavaré</a> </div> <div class="periodical"> <em>Statistics and Computing</em>, 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1007/s11222-025-10748-x" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://link.springer.com/article/10.1007/s11222-025-10748-x" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/dinh2025approximate.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1007/s11222-025-10748-x" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Approximate Bayesian Computation (ABC) is a popular inference method when likelihoods are hard to come by. Practical bottlenecks of ABC applications include selecting statistics that summarize the data without losing too much information or introducing uncertainty, and choosing distance functions and tolerance thresholds that balance accuracy and computational efficiency. Recent studies have shown that ABC methods using random forest (RF) methodology perform well while circumventing many of ABC’s drawbacks. However, RF construction is computationally expensive for large numbers of trees and model simulations, and there can be high uncertainty in the posterior if the prior distribution is uninformative. Here we further adapt random forests to the ABC setting in two ways. The first exploits distributional random forests to provide a direct method for inferring the joint posterior distribution of parameters of interest, while the second describes a sequential Monte Carlo approach which updates the prior distribution iteratively to focus on the most likely regions in the parameter space. We show that the new methods can accurately infer posterior distributions for a wide range of deterministic and stochastic models in different scientific areas.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">dinh2025approximate</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Approximate Bayesian computation sequential Monte Carlo via random forests}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Dinh, Khanh N. and Liu, C{\'e}cile and Xiang, Zijin and Liu, Zhihan and Tavar{\'e}, Simon}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Statistics and Computing}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2025}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1007/s11222-025-10748-x}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/dinh2025cinner-480.webp 480w,/assets/img/publication_preview/dinh2025cinner-800.webp 800w,/assets/img/publication_preview/dinh2025cinner-1400.webp 1400w," type="image/webp" sizes="200px"></source> <img src="/assets/img/publication_preview/dinh2025cinner.jpg" class="preview z-depth-1 rounded" width="100%" height="auto" alt="dinh2025cinner.jpg" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="dinh2025cinner" class="col-sm-8"> <div class="title">CINner: modeling and simulation of chromosomal instability in cancer at single-cell resolution</div> <div class="author"> <em>Khanh N. Dinh</em>, Ignacio Vázquez-Garcı́a, Andrew Chan, Rhea Malhotra, Adam Weiner, <a href="https://www.mskcc.org/profile/andrew-mcpherson" rel="external nofollow noopener" target="_blank">Andrew W. McPherson</a>, and <a href="https://tavarelab.cancerdynamics.columbia.edu" rel="external nofollow noopener" target="_blank">Simon Tavaré</a> </div> <div class="periodical"> <em>PLoS Computational Biology</em>, 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1371/journal.pcbi.1012902" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1012902" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/dinh2025cinner.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1371/journal.pcbi.1012902" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Cancer development is characterized by chromosomal instability, manifesting in frequent occurrences of different genomic alteration mechanisms ranging in extent and impact. Mathematical modeling can help evaluate the role of each mutational process during tumor progression, however existing frameworks can only capture certain aspects of chromosomal instability (CIN). We present CINner, a mathematical framework for modeling genomic diversity and selection during tumor evolution. The main advantage of CINner is its flexibility to incorporate many genomic events that directly impact cellular fitness, from driver gene mutations to copy number alterations (CNAs), including focal amplifications and deletions, missegregations and whole-genome duplication (WGD). We apply CINner to find chromosome-arm selection parameters that drive tumorigenesis in the absence of WGD in chromosomally unstable cancer types from the Pan-Cancer Analysis of Whole Genomes (PCAWG, n=718). We found that the selection parameters predict WGD prevalence among different chromosomally unstable tumors, hinting that the selective advantage of WGD cells hinges on their tolerance for aneuploidy and escape from nullisomy. Analysis of inference results using CINner across cancer types in The Cancer Genome Atlas (n=8207) further reveals that the inferred selection parameters reflect the bias between tumor suppressor genes and oncogenes on specific genomic regions. Direct application of CINner to model the WGD proportion and fraction of genome altered (FGA) in PCAWG uncovers the increase in CNA probabilities associated with WGD in each cancer type. CINner can also be utilized to study chromosomally stable cancer types, by applying a selection model based on driver gene mutations and focal amplifications or deletions (chronic lymphocytic leukemia in PCAWG, n=95). Finally, we used CINner to analyze the impact of CNA probabilities, chromosome selection parameters, tumor growth dynamics and population size on cancer fitness and heterogeneity. We expect that CINner will provide a powerful modeling tool for the oncology community to quantify the impact of newly uncovered genomic alteration mechanisms on shaping tumor progression and adaptation.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">dinh2025cinner</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{CINner: modeling and simulation of chromosomal instability in cancer at single-cell resolution}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Dinh, Khanh N. and V{\'a}zquez-Garc{\'\i}a, Ignacio and Chan, Andrew and Malhotra, Rhea and Weiner, Adam and McPherson, Andrew W. and Tavar{\'e}, Simon}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{PLoS Computational Biology}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2025}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1371/journal.pcbi.1012902}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/mcpherson2025ongoing-480.webp 480w,/assets/img/publication_preview/mcpherson2025ongoing-800.webp 800w,/assets/img/publication_preview/mcpherson2025ongoing-1400.webp 1400w," type="image/webp" sizes="200px"></source> <img src="/assets/img/publication_preview/mcpherson2025ongoing.jpg" class="preview z-depth-1 rounded" width="100%" height="auto" alt="mcpherson2025ongoing.jpg" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="mcpherson2025ongoing" class="col-sm-8"> <div class="title">Ongoing genome doubling shapes evolvability and immunity in ovarian cancer</div> <div class="author"> <a href="https://www.mskcc.org/profile/andrew-mcpherson" rel="external nofollow noopener" target="_blank">Andrew W. McPherson</a>, Ignacio Vázquez-Garcı́a, Matthew A. Myers, Duaa H. Al-Rawi, Matthew Zatzman, Adam C. Weiner, Samuel S. Freeman, Neeman Mohibullah, Gryte Satas, Marc J. Williams, Nicholas Ceglia, Danguolė Norbūnaitė, Allen W. Zhang, Jun Li, Jamie L. P. Lim, Michele Wu, Seongmin Choi, Eliyahu Havasov, Diljot Grewal, Hongyu Shi, Minsoo Kim, Roland F. Schwarz, Tom Kaufmann, <em>Khanh N. Dinh</em>, Florian Uhlitz, Julie Tran, Yushi Wu, Ruchi Patel, Satish Ramakrishnan, DooA Kim, Justin Clarke, Hunter Green, Emily Ali, Melody DiBona, Nancy Varice, Ritika Kundra, Vance Broach, Ginger J. Gardner, Kara L. Roche, Yukio Sonoda, Oliver Zivanovic, Sarah H. Kim, Rachel N. Grisham, Ying L. Liu, Agnes Viale, Nicole Rusk, Yulia Lakhman, Lora H. Ellenson, <a href="https://tavarelab.cancerdynamics.columbia.edu" rel="external nofollow noopener" target="_blank">Simon Tavaré</a>, Samuel Aparicio, Dennis S. Chi, Carol Aghajanian, Nadeem R. Abu-Rustum, Claire F. Friedman, Dmitriy Zamarin, Britta Weigelt, Samuel F. Bakhoum, and Sohrab P. Shah </div> <div class="periodical"> <em>Nature</em>, 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1038/s41586-025-09240-3" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.nature.com/articles/s41586-025-09240-3" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/mcpherson2025ongoing.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41586-025-09240-3" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Whole-genome doubling (WGD) is a common feature of human cancers and is linked to tumour progression, drug resistance, and metastasis. Here we examine the impact of WGD on somatic evolution and immune evasion at single-cell resolution in patient tumours. Using single-cell whole-genome sequencing, we analysed 70 high-grade serous ovarian cancer samples from 41 patients (30,260 tumour genomes) and observed near-ubiquitous evidence that WGD is an ongoing mutational process. WGD was associated with increased cell-cell diversity and higher rates of chromosomal missegregation and consequent micronucleation. We developed a mutation-based WGD timing method called doubleTime to delineate specific modes by which WGD can drive tumour evolution, including early fixation followed by considerable diversification, multiple parallel WGD events on a pre-existing background of copy-number diversity, and evolutionarily late WGD in small clones and individual cells. Furthermore, using matched single-cell RNA sequencing and high-resolution immunofluorescence microscopy, we found that inflammatory signalling and cGAS-STING pathway activation result from ongoing chromosomal instability, but this is restricted to predominantly diploid tumours (WGD-low). By contrast, predominantly WGD tumours (WGD-high), despite increased missegregation, exhibited cell-cycle dysregulation, STING1 repression, and immunosuppressive phenotypic states. Together, these findings establish WGD as an ongoing mutational process that promotes evolvability and dysregulated immunity in high-grade serous ovarian cancer.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">mcpherson2025ongoing</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Ongoing genome doubling shapes evolvability and immunity in ovarian cancer}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{McPherson, Andrew W. and V{\'a}zquez-Garc{\'\i}a, Ignacio and Myers, Matthew A. and Al-Rawi, Duaa H. and Zatzman, Matthew and Weiner, Adam C. and Freeman, Samuel S. and Mohibullah, Neeman and Satas, Gryte and Williams, Marc J. and Ceglia, Nicholas and Norb\={u}nait\.{e}, Danguol\.{e} and Zhang, Allen W. and Li, Jun and Lim, Jamie L. P. and Wu, Michele and Choi, Seongmin and Havasov, Eliyahu and Grewal, Diljot and Shi, Hongyu and Kim, Minsoo and Schwarz, Roland F. and Kaufmann, Tom and Dinh, Khanh N. and Uhlitz, Florian and Tran, Julie and Wu, Yushi and Patel, Ruchi and Ramakrishnan, Satish and Kim, DooA and Clarke, Justin and Green, Hunter and Ali, Emily and DiBona, Melody and Varice, Nancy and Kundra, Ritika and Broach, Vance and Gardner, Ginger J. and Roche, Kara L. and Sonoda, Yukio and Zivanovic, Oliver and Kim, Sarah H. and Grisham, Rachel N. and Liu, Ying L. and Viale, Agnes and Rusk, Nicole and Lakhman, Yulia and Ellenson, Lora H. and Tavar{\'e}, Simon and Aparicio, Samuel and Chi, Dennis S. and Aghajanian, Carol and Abu-Rustum, Nadeem R. and Friedman, Claire F. and Zamarin, Dmitriy and Weigelt, Britta and Bakhoum, Samuel F. and Shah, Sohrab P.}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Nature}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2025}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Nature Publishing Group UK London}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/s41586-025-09240-3}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/lee2025dysregulated-480.webp 480w,/assets/img/publication_preview/lee2025dysregulated-800.webp 800w,/assets/img/publication_preview/lee2025dysregulated-1400.webp 1400w," type="image/webp" sizes="200px"></source> <img src="/assets/img/publication_preview/lee2025dysregulated.jpg" class="preview z-depth-1 rounded" width="100%" height="auto" alt="lee2025dysregulated.jpg" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="lee2025dysregulated" class="col-sm-8"> <div class="title">Dysregulated mitochondrial energy metabolism drives the progression of mucosal field effects to invasive bladder cancer</div> <div class="author"> Sangkyou Lee, Sung Yun Jung, Pawel Kuś, Jolanta Bondaruk, June Goo Lee, <a href="https://omega.polsl.pl/info/author/PSLb8d1f042acff4c888f9e473dbc9a2448?ps=20&amp;lang=en&amp;title=&amp;pn=1&amp;cid=1686256" rel="external nofollow noopener" target="_blank">Roman Jaksik</a>, Nagireddy Putluri, <em>Khanh N. Dinh</em>, David Cogdell, Huiqin Chen, Yishan Wang, Jiansong Chen, Neema Navai, Colin Dinney, Cathy Mendelsohn, David McConkey, Richard R. Behringer, Charles C. Guo, Peng Wei, <a href="https://profiles.rice.edu/faculty/marek-kimmel" rel="external nofollow noopener" target="_blank">Marek Kimmel</a>, and Bogdan Czerniak </div> <div class="periodical"> <em>The Journal of Pathology</em>. Highlighted in <a href="https://www.nature.com/articles/s41585-025-01110-x" rel="external nofollow noopener" target="_blank">Nature Reviews Urology</a> , 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1002/path.6474" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://pathsocjournals.onlinelibrary.wiley.com/doi/10.1002/path.6474" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/lee2025dysregulated.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1002/path.6474" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Multiplatform mutational and gene expression profiling complemented with proteomic and metabolomic spatial mapping were used on the whole-organ scale to identify the molecular profile of bladder cancer evolution from field effects. Analysis of the mutational landscape identified three types of mutations, referred to as alpha, beta, and gamma. Time modeling of the mutations revealed that carcinogenesis may span 30 years and can be divided into dormant and progressive phases. The alpha mutations developed in the dormant phase. The progressive phase lasted 5 years and was signified by expanding beta mutations, but it was driven to invasive cancer by gamma mutations. The mutational landscape emerged on a background of disorganized urothelial differentiation, activated epithelial-mesenchymal transition, and enhanced immune infiltration with T-cell exhaustion. Complex dysregulation of mitochondrial energy metabolism with downregulation of oxidative phosphorylation emerged as the leading mechanism driving the progression of mucosal field effects to invasive cancer.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">lee2025dysregulated</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Dysregulated mitochondrial energy metabolism drives the progression of mucosal field effects to invasive bladder cancer}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Lee, Sangkyou and Jung, Sung Yun and Ku{\'s}, Pawel and Bondaruk, Jolanta and Lee, June Goo and Jaksik, Roman and Putluri, Nagireddy and Dinh, Khanh N. and Cogdell, David and Chen, Huiqin and Wang, Yishan and Chen, Jiansong and Navai, Neema and Dinney, Colin and Mendelsohn, Cathy and McConkey, David and Behringer, Richard R. and Guo, Charles C. and Wei, Peng and Kimmel, Marek and Czerniak, Bogdan}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{The Journal of Pathology}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2025}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Pathological Society}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1002/path.6474}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/kurpas2025statistical-480.webp 480w,/assets/img/publication_preview/kurpas2025statistical-800.webp 800w,/assets/img/publication_preview/kurpas2025statistical-1400.webp 1400w," type="image/webp" sizes="200px"></source> <img src="/assets/img/publication_preview/kurpas2025statistical.jpg" class="preview z-depth-1 rounded" width="100%" height="auto" alt="kurpas2025statistical.jpg" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="kurpas2025statistical" class="col-sm-8"> <div class="title">Statistical and evolutionary analysis of sequenced DNA from breast cancer FFPE specimens</div> <div class="author"> <a href="https://omega.polsl.pl/info/author/PSLcd398552d9ca46288a013d77ff4976bc?r=publication&amp;ps=20&amp;tab=publications&amp;lang=en&amp;title=Person%2Bprofile%2B%25E2%2580%2593%2BMonika%2BKurpas%2B%25E2%2580%2593%2BSilesian%2BUniversity%2Bof%2BTechnology&amp;pn=1&amp;cid=611238" rel="external nofollow noopener" target="_blank">Monika K. Kurpas</a>, Pawel Kuś, <a href="https://omega.polsl.pl/info/author/PSLb8d1f042acff4c888f9e473dbc9a2448?ps=20&amp;lang=en&amp;title=&amp;pn=1&amp;cid=1686256" rel="external nofollow noopener" target="_blank">Roman Jaksik</a>, <em>Khanh N. Dinh</em>, Agnieszka Adamczyk, Kaja Majchrzyk, and <a href="https://profiles.rice.edu/faculty/marek-kimmel" rel="external nofollow noopener" target="_blank">Marek Kimmel</a> </div> <div class="periodical"> <em>bioRxiv</em>, 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1101/2025.10.04.680485" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.biorxiv.org/content/10.1101/2025.10.04.680485v1" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/kurpas2025statistical.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1101/2025.10.04.680485" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Despite the introduction of instant freezing of tumor specimens, formalin-fixed paraffin-embedded (FFPE) blocks of tissue are still commonplace in clinical practice and constitute an important reference for genetic epidemiology of cancer. We carried out a study of a collection of breast tumors paired with lymph-node metastases and analyzed using advanced computational methods, to determine how much information can be obtained from mid-depth whole-exome bulk DNA sequencing. We gathered 15 paired (primary and an involved lymph node) excised breast tumors of different molecular subtypes (HER2+, triple negative, luminal A and luminal B HER2-), from the National Research Institute of Oncology, Krakow (Poland) Branch. FFPE specimens contained typical artifacts, manifesting themselves in spurious DNA variant calls. We used several bioinformatics tools to remove the artifacts and analyzed the exomic data, using both commercial and original in-house computational techniques. We used several of recent bioinformatics tools to remove the FFPE artifacts and found a serious dispersal of outcomes. After calibration, a series of analyses was performed, including copy number study, resulting in ploidy levels ranging from 1 to 5 (average of 2.5). Positive association was found between the frequency of oncogenes relative to tumor suppressor genes and DNA copy number. In addition, we carried out analyses of the clonal structure of the data using original computational methods based on evolutionary modeling. Interesting results concerning clonal structure, early tumor expansion, and interdependence of the primary tumor and lymph node metastases have been obtained. Despite the imperfections of the FFPE data, many important features of molecular evolution of tumor DNA can be recovered from routine clinical samples.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">kurpas2025statistical</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Statistical and evolutionary analysis of sequenced DNA from breast cancer FFPE specimens}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Kurpas, Monika K. and Ku{\'s}, Pawel and Jaksik, Roman and Dinh, Khanh N. and Adamczyk, Agnieszka and Majchrzyk, Kaja and Kimmel, Marek}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{bioRxiv}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2025}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1101/2025.10.04.680485}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/wesley2025wavelet-480.webp 480w,/assets/img/publication_preview/wesley2025wavelet-800.webp 800w,/assets/img/publication_preview/wesley2025wavelet-1400.webp 1400w," type="image/webp" sizes="200px"></source> <img src="/assets/img/publication_preview/wesley2025wavelet.jpg" class="preview z-depth-1 rounded" width="100%" height="auto" alt="wesley2025wavelet.jpg" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="wesley2025wavelet" class="col-sm-8"> <div class="title">Wavelet based whole genome doubling aware single cell copy number calling</div> <div class="author"> Benjamin K. Wesley, Frank Wos, Soren Germer, Jade E. B. Carter, Silas Maniatis, <em>Khanh N. Dinh</em>, Timothy Chu, Nicolas Robine, Rebecca Fitzgerald, Lizhe Zhuang, <a href="https://tavarelab.cancerdynamics.columbia.edu" rel="external nofollow noopener" target="_blank">Simon Tavaré</a>, and Karol Nowicki-Osuch </div> <div class="periodical"> <em>bioRxiv</em>, 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.64898/2025.12.18.693686" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.biorxiv.org/content/10.64898/2025.12.18.693686v1" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/wesley2025wavelet.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.64898/2025.12.18.693686" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Advances in single cell whole genome sequencing enable profiling of the copy number state of thousands of cells with little sequencing bias across the genome. The Direct Library Preparation + (DLP+) technique is a whole genome amplification-free single cell whole genome sequencing method that achieves high throughput by fragmenting each cell’s genome and ligating sequencing adapters using a modified Tn5 transposase, and sequencing to low coverage (less than 0.1x). Despite recent advances in experimental approaches, data analysis of scDNA lags behind and the existing methods are not optimized for the analysis of frozen samples with variable DNA preservation. Furthermore, existing tools predominantly rely on read depth ratio in predefined genomic bins to call copy number, making whole genome detection unidentifiable. To address this, we introduce Songbird, a single cell whole genome sequencing copy number caller that is whole genome duplication sensitive, and outperforms existing tools both in breakpoint identification and true copy number detection. We demonstrate that Songbird is robust down to extremely low coverage (less than 0.01x), adaptable to a variety of genome versions (hg19, hg38, hs.1), and is extensible to other single cell whole genome sequencing methods that rely on Tn5 tagmentation to fragment the genome.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">wesley2025wavelet</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Wavelet based whole genome doubling aware single cell copy number calling}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Wesley, Benjamin K. and Wos, Frank and Germer, Soren and Carter, Jade E. B. and Maniatis, Silas and Dinh, Khanh N. and Chu, Timothy and Robine, Nicolas and Fitzgerald, Rebecca and Zhuang, Lizhe and Tavar{\'e}, Simon and Nowicki-Osuch, Karol}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{bioRxiv}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2025}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.64898/2025.12.18.693686}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> </ol> <h2 class="bibliography">2024</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/dinh2024comparison-480.webp 480w,/assets/img/publication_preview/dinh2024comparison-800.webp 800w,/assets/img/publication_preview/dinh2024comparison-1400.webp 1400w," type="image/webp" sizes="200px"></source> <img src="/assets/img/publication_preview/dinh2024comparison.jpg" class="preview z-depth-1 rounded" width="100%" height="auto" alt="dinh2024comparison.jpg" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="dinh2024comparison" class="col-sm-8"> <div class="title">Comparison of tug-of-war models assuming Moran versus branching process population dynamics</div> <div class="author"> <em>Khanh N. Dinh</em>, <a href="https://omega.polsl.pl/info/author/PSLcd398552d9ca46288a013d77ff4976bc?r=publication&amp;ps=20&amp;tab=publications&amp;lang=en&amp;title=Person%2Bprofile%2B%25E2%2580%2593%2BMonika%2BKurpas%2B%25E2%2580%2593%2BSilesian%2BUniversity%2Bof%2BTechnology&amp;pn=1&amp;cid=611238" rel="external nofollow noopener" target="_blank">Monika K. Kurpas</a>, and <a href="https://profiles.rice.edu/faculty/marek-kimmel" rel="external nofollow noopener" target="_blank">Marek Kimmel</a> </div> <div class="periodical"> <em>eLife</em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.7554/eLife.94597.1" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://elifesciences.org/reviewed-preprints/94597" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/dinh2024comparison.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.7554/eLife.94597.1" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Mutations arising during cancer evolution are typically categorized as either ’drivers’ or ’passengers’, depending on whether they increase the cell fitness. Recently, McFarland et al. introduced the Tug-of-War model for the joint effect of rare advantageous drivers and frequent but deleterious passengers. We examine this model under two common but distinct frameworks, the Moran model and the branching process. We show that frequently used statistics are similar between a version of the Moran model and the branching process conditioned on the final cell count, under different selection scenarios. We infer the selection coefficients for three breast cancer samples, resulting in good fits of the shape of their Site Frequency Spectra. All fitted values for the selective disadvantage of passenger mutations are nonzero, supporting the view that they exert deleterious selection during tumorigenesis that driver mutations must compensate.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">dinh2024comparison</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Comparison of tug-of-war models assuming Moran versus branching process population dynamics}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Dinh, Khanh N. and Kurpas, Monika K. and Kimmel, Marek}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{eLife}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{13}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{eLife Sciences Publications Limited}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.7554/eLife.94597.1}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/jaksik2024multiomics-480.webp 480w,/assets/img/publication_preview/jaksik2024multiomics-800.webp 800w,/assets/img/publication_preview/jaksik2024multiomics-1400.webp 1400w," type="image/webp" sizes="200px"></source> <img src="/assets/img/publication_preview/jaksik2024multiomics.jpg" class="preview z-depth-1 rounded" width="100%" height="auto" alt="jaksik2024multiomics.jpg" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="jaksik2024multiomics" class="col-sm-8"> <div class="title">Multiomics-based feature extraction and selection for the prediction of lung cancer survival</div> <div class="author"> <a href="https://omega.polsl.pl/info/author/PSLb8d1f042acff4c888f9e473dbc9a2448?ps=20&amp;lang=en&amp;title=&amp;pn=1&amp;cid=1686256" rel="external nofollow noopener" target="_blank">Roman Jaksik</a>, Kamila Szumała, <em>Khanh N. Dinh</em>, and Jarosław Śmieja </div> <div class="periodical"> <em>International Journal of Molecular Sciences</em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.3390/ijms25073661" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.mdpi.com/1422-0067/25/7/3661" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/jaksik2024multiomics.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.3390/ijms25073661" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Lung cancer is a global health challenge, hindered by delayed diagnosis and the disease’s complex molecular landscape. Accurate patient survival prediction is critical, motivating the exploration of various -omics datasets using machine learning methods. Leveraging multi-omics data, this study seeks to enhance the accuracy of survival prediction by proposing new feature extraction techniques combined with unbiased feature selection. Two lung adenocarcinoma multi-omics datasets, originating from the TCGA and CPTAC-3 projects, were employed for this purpose, emphasizing gene expression, methylation, and mutations as the most relevant data sources that provide features for the survival prediction models. Additionally, gene set aggregation was shown to be the most effective feature extraction method for mutation and copy number variation data. Using the TCGA dataset, we identified 32 molecular features that allowed the construction of a 2-year survival prediction model with an AUC of 0.839. The selected features were additionally tested on an independent CPTAC-3 dataset, achieving an AUC of 0.815 in nested cross-validation, which confirmed the robustness of the identified features.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">jaksik2024multiomics</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Multiomics-based feature extraction and selection for the prediction of lung cancer survival}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Jaksik, Roman and Szuma{\l}a, Kamila and Dinh, Khanh N. and {\'S}mieja, Jaros{\l}aw}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{International Journal of Molecular Sciences}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{25}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{7}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{3661}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{MDPI}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.3390/ijms25073661}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/xiang2024inference-480.webp 480w,/assets/img/publication_preview/xiang2024inference-800.webp 800w,/assets/img/publication_preview/xiang2024inference-1400.webp 1400w," type="image/webp" sizes="200px"></source> <img src="/assets/img/publication_preview/xiang2024inference.jpg" class="preview z-depth-1 rounded" width="100%" height="auto" alt="xiang2024inference.jpg" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="xiang2024inference" class="col-sm-8"> <div class="title">Inference of chromosome selection parameters and missegregation rate in cancer from DNA-sequencing data</div> <div class="author"> Zijin Xiang, Zhihan Liu, and <em>Khanh N. Dinh</em> </div> <div class="periodical"> <em>Scientific Reports</em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1038/s41598" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.nature.com/articles/s41598-024-67842-9" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/xiang2024inference.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41598" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Aneuploidy is frequently observed in cancers and has been linked to poor patient outcome. Analysis of aneuploidy in DNA-sequencing (DNA-seq) data necessitates untangling the effects of the Copy Number Aberration (CNA) occurrence rates and the selection coefficients that act upon the resulting karyotypes. We introduce a parameter inference algorithm that takes advantage of both bulk and single-cell DNA-seq cohorts. The method is based on Approximate Bayesian Computation (ABC) and utilizes CINner, our recently introduced simulation algorithm of chromosomal instability in cancer. We examine three groups of statistics to summarize the data in the ABC routine: (A) Copy Number-based measures, (B) phylogeny tip statistics, and (C) phylogeny balance indices. Using these statistics, our method can recover both the CNA probabilities and selection parameters from ground truth data, and performs well even for data cohorts of relatively small sizes. We find that only statistics in groups A and C are well-suited for identifying CNA probabilities, and only group A carries the signals for estimating selection parameters. Moreover, the low number of CNA events at large scale compared to cell counts in single-cell samples means that statistics in group B cannot be estimated accurately using phylogeny reconstruction algorithms at the chromosome level. As data from both bulk and single-cell DNA-sequencing techniques becomes increasingly available, our inference framework promises to facilitate the analysis of distinct cancer types, differentiation between selection and neutral drift, and prediction of cancer clonal dynamics.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">xiang2024inference</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Inference of chromosome selection parameters and missegregation rate in cancer from DNA-sequencing data}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Xiang, Zijin and Liu, Zhihan and Dinh, Khanh N.}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Scientific Reports}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{14}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{17699}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Nature Publishing Group UK London}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/s41598}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> </ol> <h2 class="bibliography">2022</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/bondaruk2022origin-480.webp 480w,/assets/img/publication_preview/bondaruk2022origin-800.webp 800w,/assets/img/publication_preview/bondaruk2022origin-1400.webp 1400w," type="image/webp" sizes="200px"></source> <img src="/assets/img/publication_preview/bondaruk2022origin.jpg" class="preview z-depth-1 rounded" width="100%" height="auto" alt="bondaruk2022origin.jpg" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="bondaruk2022origin" class="col-sm-8"> <div class="title">The origin of bladder cancer from mucosal field effects</div> <div class="author"> Jolanta Bondaruk, <a href="https://omega.polsl.pl/info/author/PSLb8d1f042acff4c888f9e473dbc9a2448?ps=20&amp;lang=en&amp;title=&amp;pn=1&amp;cid=1686256" rel="external nofollow noopener" target="_blank">Roman Jaksik</a>, Ziqiao Wang, David Cogdell, Sangkyou Lee, Yujie Chen, <em>Khanh N. Dinh</em>, Tadeusz Majewski, Li Zhang, Shaolong Cao, Feng Tian, Hui Yao, Pawel Kus, Huiqin Chen, John N. Weinstein, Neema Navai, Colin Dinney, Jianjun Gao, Dan Theodorescu, Christopher Logothetis, Charles C. Guo, Wenyi Wang, David McConkey, Peng Wei, <a href="https://profiles.rice.edu/faculty/marek-kimmel" rel="external nofollow noopener" target="_blank">Marek Kimmel</a>, and Bogdan Czerniak </div> <div class="periodical"> <em>iScience</em>, 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1016/j.isci.2022.104551" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.sciencedirect.com/science/article/pii/S2589004222008239?via%3Dihub" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/bondaruk2022origin.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.isci.2022.104551" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Whole-organ mapping was used to study molecular changes in the evolution of bladder cancer from field effects. We identified more than 100 dysregulated pathways, involving immunity, differentiation, and transformation, as initiators of carcinogenesis. Dysregulation of interleukins signified the involvement of inflammation in the incipient phases of the process. An aberrant methylation/expression of multiple HOX genes signified dysregulation of the differentiation program. We identified three types of mutations based on their geographic distribution. The most common were mutations restricted to individual mucosal samples that targeted uroprogenitor cells. Two types of mutations were associated with clonal expansion and involved large areas of mucosa. The αmutations occurred at low frequencies while the βmutations increased in frequency with disease progression. Modeling revealed that bladder carcinogenesis spans 10-15 years and can be divided into dormant and progressive phases. The progressive phase lasted 1-2 years and was driven by βmutations.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">bondaruk2022origin</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{The origin of bladder cancer from mucosal field effects}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Bondaruk, Jolanta and Jaksik, Roman and Wang, Ziqiao and Cogdell, David and Lee, Sangkyou and Chen, Yujie and Dinh, Khanh N. and Majewski, Tadeusz and Zhang, Li and Cao, Shaolong and Tian, Feng and Yao, Hui and Kus, Pawel and Chen, Huiqin and Weinstein, John N. and Navai, Neema and Dinney, Colin and Gao, Jianjun and Theodorescu, Dan and Logothetis, Christopher and Guo, Charles C. and Wang, Wenyi and McConkey, David and Wei, Peng and Kimmel, Marek and Czerniak, Bogdan}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{iScience}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{25}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{7}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2022}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Elsevier}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.isci.2022.104551}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/funnell2022single-480.webp 480w,/assets/img/publication_preview/funnell2022single-800.webp 800w,/assets/img/publication_preview/funnell2022single-1400.webp 1400w," type="image/webp" sizes="200px"></source> <img src="/assets/img/publication_preview/funnell2022single.jpg" class="preview z-depth-1 rounded" width="100%" height="auto" alt="funnell2022single.jpg" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="funnell2022single" class="col-sm-8"> <div class="title">Single-cell genomic variation induced by mutational processes in cancer</div> <div class="author"> Tyler Funnell, Ciara H O’Flanagan, Marc J Williams, <a href="https://www.mskcc.org/profile/andrew-mcpherson" rel="external nofollow noopener" target="_blank">Andrew McPherson</a>, Steven McKinney, Farhia Kabeer, Hakwoo Lee, Sohrab Salehi, Ignacio Vázquez-Garcı́a, Hongyu Shi, Emily Leventhal, Tehmina Masud, Peter Eirew, Damian Yap, Allen W. Zhang, Jamie L. P. Lim, Beixi Wang, Jazmine Brimhall, Justina Biele, Jerome Ting, Vinci Au, Michael Van Vliet, Yi Fei Liu, Sean Beatty, Daniel Lai, Jenifer Pham, Diljot Grewal, Douglas Abrams, Eliyahu Havasov, Samantha Leung, Viktoria Bojilova, Richard A. Moore, Nicole Rusk, Florian Uhlitz, Nicholas Ceglia, Adam C. Weiner, Elena Zaikova, J. Maxwell Douglas, Dmitriy Zamarin, Britta Weigelt, Sarah H. Kim, Arnaud Da Cruz Paula, Jorge S. Reis-Filho, Spencer D. Martin, Yangguang Li, Hong Xu, Teresa Ruiz Algara, So Ra Lee, Viviana Cerda Llanos, David G. Huntsman, Jessica N. McAlpine, IMAXT Consortium, Sohrab P. Shah, and Samuel Aparicio <i class="fa-solid fa-circle-info ml-1" data-toggle="popover" data-placement="top" data-html="true" data-content="As part of the IMAXT Consortium"> </i> </div> <div class="periodical"> <em>Nature</em>, 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1038/s41586-022-05249-0" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.nature.com/articles/s41586-022-05249-0" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/funnell2022single.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41586-022-05249-0" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>How cell-to-cell copy number alterations that underpin genomic instability in human cancers drive genomic and phenotypic variation, and consequently the evolution of cancer, remains understudied. Here, by applying scaled single-cell whole-genome sequencing to wild-type, TP53-deficient and TP53-deficient; BRCA1-deficient or TP53-deficient; BRCA2-deficient mammary epithelial cells (13,818 genomes), and to primary triple-negative breast cancer (TNBC) and high-grade serous ovarian cancer (HGSC) cells (22,057 genomes), we identify three distinct ’foreground’ mutational patterns that are defined by cell-to-cell structural variation. Cell- and clone-specific high-level amplifications, parallel haplotype-specific copy number alterations and copy number segment length variation (serrate structural variations) had measurable phenotypic and evolutionary consequences. In TNBC and HGSC, clone-specific high-level amplifications in known oncogenes were highly prevalent in tumours bearing fold-back inversions, relative to tumours with homologous recombination deficiency, and were associated with increased clone-to-clone phenotypic variation. Parallel haplotype-specific alterations were also commonly observed, leading to phylogenetic evolutionary diversity and clone-specific mono-allelic expression. Serrate variants were increased in tumours with fold-back inversions and were highly correlated with increased genomic diversity of cellular populations. Together, our findings show that cell-to-cell structural variation contributes to the origins of phenotypic and evolutionary diversity in TNBC and HGSC, and provide insight into the genomic and mutational states of individual cancer cells.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">funnell2022single</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Single-cell genomic variation induced by mutational processes in cancer}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Funnell, Tyler and O'Flanagan, Ciara H and Williams, Marc J and McPherson, Andrew and McKinney, Steven and Kabeer, Farhia and Lee, Hakwoo and Salehi, Sohrab and V{\'a}zquez-Garc{\'\i}a, Ignacio and Shi, Hongyu and Leventhal, Emily and Masud, Tehmina and Eirew, Peter and Yap, Damian and Zhang, Allen W. and Lim, Jamie L. P. and Wang, Beixi and Brimhall, Jazmine and Biele, Justina and Ting, Jerome and Au, Vinci and Van Vliet, Michael and Liu, Yi Fei and Beatty, Sean and Lai, Daniel and Pham, Jenifer and Grewal, Diljot and Abrams, Douglas and Havasov, Eliyahu and Leung, Samantha and Bojilova, Viktoria and Moore, Richard A. and Rusk, Nicole and Uhlitz, Florian and Ceglia, Nicholas and Weiner, Adam C. and Zaikova, Elena and Douglas, J. Maxwell and Zamarin, Dmitriy and Weigelt, Britta and Kim, Sarah H. and Da Cruz Paula, Arnaud and Reis-Filho, Jorge S. and Martin, Spencer D. and Li, Yangguang and Xu, Hong and de Algara, Teresa Ruiz and Lee, So Ra and Llanos, Viviana Cerda and Huntsman, David G. and McAlpine, Jessica N. and Consortium, IMAXT and Shah, Sohrab P. and Aparicio, Samuel}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Nature}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{612}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{7938}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{106--115}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2022}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Nature Publishing Group UK London}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/s41586-022-05249-0}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> </ol> <h2 class="bibliography">2021</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/dinh2021predicting-480.webp 480w,/assets/img/publication_preview/dinh2021predicting-800.webp 800w,/assets/img/publication_preview/dinh2021predicting-1400.webp 1400w," type="image/webp" sizes="200px"></source> <img src="/assets/img/publication_preview/dinh2021predicting.jpg" class="preview z-depth-1 rounded" width="100%" height="auto" alt="dinh2021predicting.jpg" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="dinh2021predicting" class="col-sm-8"> <div class="title">Predicting time to relapse in acute myeloid leukemia through stochastic modeling of minimal residual disease based on clonality data</div> <div class="author"> <em>Khanh N. Dinh</em>, <a href="https://omega.polsl.pl/info/author/PSLb8d1f042acff4c888f9e473dbc9a2448?ps=20&amp;lang=en&amp;title=&amp;pn=1&amp;cid=1686256" rel="external nofollow noopener" target="_blank">Roman Jaksik</a>, <a href="https://www.lerner.ccf.org/cancer-biology/corey/" rel="external nofollow noopener" target="_blank">Seth J. Corey</a>, and <a href="https://profiles.rice.edu/faculty/marek-kimmel" rel="external nofollow noopener" target="_blank">Marek Kimmel</a> </div> <div class="periodical"> <em>Computational and systems oncology</em>, 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1002/cso2.1026" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://onlinelibrary.wiley.com/doi/full/10.1002/cso2.1026" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/dinh2021predicting.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1002/cso2.1026" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Event-free and overall survival remain poor for patients with acute myeloid leukemia. Chemoresistant clones contributing to relapse arise from minimal residual disease (MRD) or newly acquired mutations. However, the dynamics of clones comprising MRD is poorly understood. We developed a predictive stochastic model, based on a multitype age-dependent Markov branching process, to describe how random events in MRD contribute to the heterogeneity in treatment response. We employed training and validation sets of patients who underwent whole-genome sequencing and for whom mutant clone frequencies at diagnosis and relapse were available. The disease evolution and treatment outcome are subject to stochastic fluctuations. Estimates of malignant clone growth rates, obtained by model fitting, are consistent with published data. Using the estimates from the training set, we developed a function linking MRD and time of relapse with MRD inferred from the model fits to clone frequencies and other data. An independent validation set confirmed our model. In a third dataset, we fitted the model to data at diagnosis and remission and predicted the time to relapse. As a conclusion, given bone marrow genome at diagnosis and MRD at or past remission, the model can predict time to relapse and help guide treatment decisions to mitigate relapse.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">dinh2021predicting</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Predicting time to relapse in acute myeloid leukemia through stochastic modeling of minimal residual disease based on clonality data}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Dinh, Khanh N. and Jaksik, Roman and Corey, Seth J. and Kimmel, Marek}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Computational and systems oncology}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{3}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{e1026}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Wiley Online Library}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1002/cso2.1026}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/salehi2021clonal-480.webp 480w,/assets/img/publication_preview/salehi2021clonal-800.webp 800w,/assets/img/publication_preview/salehi2021clonal-1400.webp 1400w," type="image/webp" sizes="200px"></source> <img src="/assets/img/publication_preview/salehi2021clonal.jpg" class="preview z-depth-1 rounded" width="100%" height="auto" alt="salehi2021clonal.jpg" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="salehi2021clonal" class="col-sm-8"> <div class="title">Clonal fitness inferred from time-series modelling of single-cell cancer genomes</div> <div class="author"> Sohrab Salehi, Farhia Kabeer, Nicholas Ceglia, Mirela Andronescu, Marc J Williams, Kieran R Campbell, Tehmina Masud, Beixi Wang, Justina Biele, Jazmine Brimhall, David Gee, Hakwoo Lee, Jerome Ting, Allen W. Zhang, Hoa Tran, Ciara O’Flanagan, Fatemeh Dorri, Nicole Rusk, Teresa Ruiz Algara, So Ra Lee, Brian Yu Chieh Cheng, Peter Eirew, Takako Kono, Jenifer Pham, Diljot Grewal, Daniel Lai, Richard Moore, Andrew J. Mungall, Marco A. Marra, IMAXT Consortium, <a href="https://www.mskcc.org/profile/andrew-mcpherson" rel="external nofollow noopener" target="_blank">Andrew McPherson</a>, Alexandre Bouchard-Côté, Samuel Aparicio, and Sohrab P. Shah <i class="fa-solid fa-circle-info ml-1" data-toggle="popover" data-placement="top" data-html="true" data-content="As part of the IMAXT Consortium"> </i> </div> <div class="periodical"> <em>Nature</em>, 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1038/s41586-021-03648-3" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.nature.com/articles/s41586-021-03648-3" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/salehi2021clonal.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41586-021-03648-3" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Progress in defining genomic fitness landscapes in cancer, especially those defined by copy number alterations (CNAs), has been impeded by lack of time-series single-cell sampling of polyclonal populations and temporal statistical models. Here we generated 42,000 genomes from multi-year time-series single-cell whole-genome sequencing of breast epithelium and primary triple-negative breast cancer (TNBC) patient-derived xenografts (PDXs), revealing the nature of CNA-defined clonal fitness dynamics induced by TP53 mutation and cisplatin chemotherapy. Using a new Wright-Fisher population genetics model to infer clonal fitness, we found that TP53 mutation alters the fitness landscape, reproducibly distributing fitness over a larger number of clones associated with distinct CNAs. Furthermore, in TNBC PDX models with mutated TP53, inferred fitness coefficients from CNA-based genotypes accurately forecast experimentally enforced clonal competition dynamics. Drug treatment in three long-term serially passaged TNBC PDXs resulted in cisplatin-resistant clones emerging from low-fitness phylogenetic lineages in the untreated setting. Conversely, high-fitness clones from treatment-naive controls were eradicated, signalling an inversion of the fitness landscape. Finally, upon release of drug, selection pressure dynamics were reversed, indicating a fitness cost of treatment resistance. Together, our findings define clonal fitness linked to both CNA and therapeutic resistance in polyclonal tumours.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">salehi2021clonal</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Clonal fitness inferred from time-series modelling of single-cell cancer genomes}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Salehi, Sohrab and Kabeer, Farhia and Ceglia, Nicholas and Andronescu, Mirela and Williams, Marc J and Campbell, Kieran R and Masud, Tehmina and Wang, Beixi and Biele, Justina and Brimhall, Jazmine and Gee, David and Lee, Hakwoo and Ting, Jerome and Zhang, Allen W. and Tran, Hoa and O’Flanagan, Ciara and Dorri, Fatemeh and Rusk, Nicole and de Algara, Teresa Ruiz and Lee, So Ra and Cheng, Brian Yu Chieh and Eirew, Peter and Kono, Takako and Pham, Jenifer and Grewal, Diljot and Lai, Daniel and Moore, Richard and Mungall, Andrew J. and Marra, Marco A. and Consortium, IMAXT and McPherson, Andrew and Bouchard-C{\o}t{\'e}, Alexandre and Aparicio, Samuel and Shah, Sohrab P.}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Nature}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{595}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{7868}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{585--590}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Nature Publishing Group UK London}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/s41586-021-03648-3}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> </ol> <h2 class="bibliography">2020</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/dinh2020application-480.webp 480w,/assets/img/publication_preview/dinh2020application-800.webp 800w,/assets/img/publication_preview/dinh2020application-1400.webp 1400w," type="image/webp" sizes="200px"></source> <img src="/assets/img/publication_preview/dinh2020application.jpg" class="preview z-depth-1 rounded" width="100%" height="auto" alt="dinh2020application.jpg" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="dinh2020application" class="col-sm-8"> <div class="title">Application of the Moran model in estimating selection coefficient of mutated CSF3R clones in the evolution of severe congenital neutropenia to myeloid neoplasia</div> <div class="author"> <em>Khanh N. Dinh</em>, <a href="https://www.lerner.ccf.org/cancer-biology/corey/" rel="external nofollow noopener" target="_blank">Seth J. Corey</a>, and <a href="https://profiles.rice.edu/faculty/marek-kimmel" rel="external nofollow noopener" target="_blank">Marek Kimmel</a> </div> <div class="periodical"> <em>Frontiers in Physiology</em>, 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.3389/fphys.2020.00806" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.frontiersin.org/journals/physiology/articles/10.3389/fphys.2020.00806/full" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/dinh2020application.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.3389/fphys.2020.00806" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Bone marrow failure (BMF) syndromes, such as severe congenital neutropenia (SCN) are leukemia predisposition syndromes. We focus here on the transition from SCN to pre-leukemic myelodysplastic syndrome (MDS). Stochastic mathematical models have been conceived that attempt to explain the transition of SCN to MDS, in the most parsimonious way, using extensions of standard processes of population genetics and population dynamics, such as the branching and the Moran processes. We previously presented a hypothesis of the SCN to MDS transition, which involves directional selection and recurrent mutation, to explain the distribution of ages at onset of MDS or AML. Based on experimental and clinical data and a model of human hematopoiesis, a range of probable values of the selection coefficient s and mutation rate μ have been determined. These estimates lead to predictions of the age at onset of MDS or AML, which are consistent with the clinical data. In the current paper, based on data extracted from published literature, we seek to provide an independent validation of these estimates. We proceed with two purposes in mind: (i) to determine the ballpark estimates of the selection coefficients and verify their consistency with those previously obtained and (ii) to provide possible insight into the role of recurrent mutations of the G-CSF receptor in the SCN to MDS transition.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">dinh2020application</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Application of the Moran model in estimating selection coefficient of mutated CSF3R clones in the evolution of severe congenital neutropenia to myeloid neoplasia}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Dinh, Khanh N. and Corey, Seth J. and Kimmel, Marek}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Frontiers in Physiology}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{11}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{806}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2020}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Frontiers Media SA}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.3389/fphys.2020.00806}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/dinh2020statistical-480.webp 480w,/assets/img/publication_preview/dinh2020statistical-800.webp 800w,/assets/img/publication_preview/dinh2020statistical-1400.webp 1400w," type="image/webp" sizes="200px"></source> <img src="/assets/img/publication_preview/dinh2020statistical.jpg" class="preview z-depth-1 rounded" width="100%" height="auto" alt="dinh2020statistical.jpg" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="dinh2020statistical" class="col-sm-8"> <div class="title">Statistical inference for the evolutionary history of cancer genomes</div> <div class="author"> <em>Khanh N. Dinh</em>, <a href="https://omega.polsl.pl/info/author/PSLb8d1f042acff4c888f9e473dbc9a2448?ps=20&amp;lang=en&amp;title=&amp;pn=1&amp;cid=1686256" rel="external nofollow noopener" target="_blank">Roman Jaksik</a>, <a href="https://profiles.rice.edu/faculty/marek-kimmel" rel="external nofollow noopener" target="_blank">Marek Kimmel</a>, <a href="https://smile.cnrs.fr/people/amaury/" rel="external nofollow noopener" target="_blank">Amaury Lambert</a>, and <a href="https://tavarelab.cancerdynamics.columbia.edu" rel="external nofollow noopener" target="_blank">Simon Tavaré</a> </div> <div class="periodical"> <em>Statistical Science</em>, 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1214/19-sts7561" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://projecteuclid.org/journals/statistical-science/volume-35/issue-1/Statistical-Inference-for-the-Evolutionary-History-of-Cancer-Genomes/10.1214/19-STS7561.full" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/dinh2020statistical.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1214/19-sts7561" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Recent years have seen considerable work on inference about cancer evolution from mutations identified in cancer samples. Much of the modeling work has been based on classical models of population genetics, generalized to accommodate time-varying cell population size. Reverse-time, genealogical views of such models, commonly known as coalescents, have been used to infer aspects of the past of growing populations. Another approach is to use branching processes, the simplest scenario being the classical linear birth-death process. Inference from evolutionary models of DNA often exploits summary statistics of the sequence data, a common one being the so-called Site Frequency Spectrum (SFS). In a bulk tumor sequencing experiment, we can estimate for each site at which a novel somatic point mutation has arisen, the proportion of cells that carry that mutation. These numbers are then grouped into collections of sites which have similar mutant fractions. We examine how the SFS based on birth-death processes differs from those based on the coalescent model. This may stem from the different sampling mechanisms in the two approaches. However, we also show that despite this, they are quantitatively comparable for the range of parameters typical for tumor cell populations. We also present a model of tumor evolution with selective sweeps, and demonstrate how it may help in understanding the history of a tumor as well as the influence of data pre-processing. We illustrate the theory with applications to several examples from The Cancer Genome Atlas tumors.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">dinh2020statistical</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Statistical inference for the evolutionary history of cancer genomes}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Dinh, Khanh N. and Jaksik, Roman and Kimmel, Marek and Lambert, Amaury and Tavar{\'e}, Simon}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Statistical Science}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{35}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{129--144}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2020}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{JSTOR}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1214/19-sts7561}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> </ol> <h2 class="bibliography">2019</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/laks2019clonal-480.webp 480w,/assets/img/publication_preview/laks2019clonal-800.webp 800w,/assets/img/publication_preview/laks2019clonal-1400.webp 1400w," type="image/webp" sizes="200px"></source> <img src="/assets/img/publication_preview/laks2019clonal.jpg" class="preview z-depth-1 rounded" width="100%" height="auto" alt="laks2019clonal.jpg" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="laks2019clonal" class="col-sm-8"> <div class="title">Clonal decomposition and DNA replication states defined by scaled single-cell genome sequencing</div> <div class="author"> Emma Laks, <a href="https://www.mskcc.org/profile/andrew-mcpherson" rel="external nofollow noopener" target="_blank">Andrew McPherson</a>, Hans Zahn, Daniel Lai, Adi Steif, Jazmine Brimhall, Justina Biele, Beixi Wang, Tehmina Masud, Jerome Ting, Diljot Grewal, Cydney Nielsen, Samantha Leung, Viktoria Bojilova, Maia Smith, Oleg Golovko, Steven Poon, Peter Eirew, Farhia Kabeer, Teresa Algara, So Ra Lee, M. Jafar Taghiyar, Curtis Huebner, Jessica Ngo, Tim Chan, Spencer Vatrt-Watts, Pascale Walters, Nafis Abrar, Sophia Chan, Matt Wiens, Lauren Martin, R. Wilder Scott, T. Michael Underhill, Elizabeth Chavez, Christian Steidl, Daniel Da Costa, Yussanne Ma, Robin J.N. Coope, Richard Corbett, Stephen Pleasance, Richard Moore, Andrew J. Mungall, Colin Mar, Fergus Cafferty, Karen Gelmon, Stephen Chia, IMAXT Consortium, Marco A. Marra, Carl Hansen, Sohrab P. Shah, and Samuel Aparicio <i class="fa-solid fa-circle-info ml-1" data-toggle="popover" data-placement="top" data-html="true" data-content="As part of the IMAXT Consortium"> </i> </div> <div class="periodical"> <em>Cell</em>, 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1016/j.cell.2019.10.026" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.cell.com/cell/fulltext/S0092-8674(19)31176-6" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/laks2019clonal.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.cell.2019.10.026" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Accurate measurement of clonal genotypes, mutational processes, and replication states from individual tumor-cell genomes will facilitate improved understanding of tumor evolution. We have developed DLP+, a scalable single-cell whole-genome sequencing platform implemented using commodity instruments, image-based object recognition, and open source computational methods. Using DLP+, we have generated a resource of 51,926 single-cell genomes and matched cell images from diverse cell types including cell lines, xenografts, and diagnostic samples with limited material. From this resource we have defined variation in mitotic mis-segregation rates across tissue types and genotypes. Analysis of matched genomic and image measurements revealed correlations between cellular morphology and genome ploidy states. Aggregation of cells sharing copy number profiles allowed for calculation of single-nucleotide resolution clonal genotypes and inference of clonal phylogenies and avoided the limitations of bulk deconvolution. Finally, joint analysis over the above features defined clone-specific chromosomal aneuploidy in polyclonal populations.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">laks2019clonal</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Clonal decomposition and DNA replication states defined by scaled single-cell genome sequencing}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Laks, Emma and McPherson, Andrew and Zahn, Hans and Lai, Daniel and Steif, Adi and Brimhall, Jazmine and Biele, Justina and Wang, Beixi and Masud, Tehmina and Ting, Jerome and Grewal, Diljot and Nielsen, Cydney and Leung, Samantha and Bojilova, Viktoria and Smith, Maia and Golovko, Oleg and Poon, Steven and Eirew, Peter and Kabeer, Farhia and Ruiz de Algara, Teresa and Lee, So Ra and Taghiyar, M. Jafar and Huebner, Curtis and Ngo, Jessica and Chan, Tim and Vatrt-Watts, Spencer and Walters, Pascale and Abrar, Nafis and Chan, Sophia and Wiens, Matt and Martin, Lauren and Scott, R. Wilder and Underhill, T. Michael and Chavez, Elizabeth and Steidl, Christian and Da Costa, Daniel and Ma, Yussanne and Coope, Robin J.N. and Corbett, Richard and Pleasance, Stephen and Moore, Richard and Mungall, Andrew J. and Mar, Colin and Cafferty, Fergus and Gelmon, Karen and Chia, Stephen and Consortium, IMAXT and Marra, Marco A. and Hansen, Carl and Shah, Sohrab P. and Aparicio, Samuel}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Cell}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{179}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{5}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1207--1221}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2019}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Elsevier}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.cell.2019.10.026}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li></ol> <h2 class="bibliography">2018</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/dinh2018comparison-480.webp 480w,/assets/img/publication_preview/dinh2018comparison-800.webp 800w,/assets/img/publication_preview/dinh2018comparison-1400.webp 1400w," type="image/webp" sizes="200px"></source> <img src="/assets/img/publication_preview/dinh2018comparison.jpg" class="preview z-depth-1 rounded" width="100%" height="auto" alt="dinh2018comparison.jpg" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="dinh2018comparison" class="col-sm-8"> <div class="title">A comparison of the Magnus expansion and other solvers for the chemical master equation with variable rates</div> <div class="author"> <em>Khanh N. Dinh</em>, and <a href="https://math.ua.edu/people/roger-b-sidje/" rel="external nofollow noopener" target="_blank">Roger B. Sidje</a> </div> <div class="periodical"> <em>In Recent Advances in Mathematical and Statistical Methods: IV AMMCS International Conference, Waterloo, Canada, August 20–25, 2017 IV</em>, 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1007/978-3-319-99719-3_24" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://link.springer.com/chapter/10.1007/978-3-319-99719-3_24" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/dinh2018comparison.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1007/978-3-319-99719-3_24" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Many traditional approaches for solving the chemical master equation (CME) cannot be used in their basic form when reaction rates change over time, for instance due to cell volume or temperature. One technique is to use the Magnus expansion to represent the solution to the CME as the action of a matrix exponential, for which Krylov-based approximation methods can be applied. In this paper, we compare two variants of the Magnus scheme with some popular ordinary differential equations (ODE) solvers, such as Adams-Bashforth, Runge-Kutta and Backward-differentiation formula (BDF). Our numerical tests show that the Magnus variants are remarkably efficient at computing the transient probability distributions of a transcriptional regulatory system where propensities vary over time due to cell volume increase.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@inproceedings</span><span class="p">{</span><span class="nl">dinh2018comparison</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{A comparison of the Magnus expansion and other solvers for the chemical master equation with variable rates}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Dinh, Khanh N. and Sidje, Roger B.}</span><span class="p">,</span>
  <span class="na">booktitle</span> <span class="p">=</span> <span class="s">{Recent Advances in Mathematical and Statistical Methods: IV AMMCS International Conference, Waterloo, Canada, August 20--25, 2017 IV}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{261--270}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2018}</span><span class="p">,</span>
  <span class="na">organization</span> <span class="p">=</span> <span class="s">{Springer}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1007/978-3-319-99719-3_24}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/dinh2018inexact-480.webp 480w,/assets/img/publication_preview/dinh2018inexact-800.webp 800w,/assets/img/publication_preview/dinh2018inexact-1400.webp 1400w," type="image/webp" sizes="200px"></source> <img src="/assets/img/publication_preview/dinh2018inexact.jpg" class="preview z-depth-1 rounded" width="100%" height="auto" alt="dinh2018inexact.jpg" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="dinh2018inexact" class="col-sm-8"> <div class="title">Inexact methods for the chemical master equation with constant or time-varying propensities, and application to parameter inference</div> <div class="author"> <em>Khanh N. Dinh</em> </div> <div class="periodical"> Doctoral thesis , 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://ir.ua.edu/items/1eec31c0-edf6-4e74-8e32-e082329f63c3" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/dinh2018inexact.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-pmid="" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Complex reaction networks arise in molecular biology and many other different ﬁelds of science such as ecology and social study. A familiar approach to modeling such problems is to ﬁnd their master equation. In systems biology, the equation is called the chemical master equation (CME), and solving the CME is a difficult task, because of the curse of dimensionality. The goal of this dissertation is to alleviate this curse via the use of the ﬁnite state projection (FSP), in both cases where the CME matrix is constant (if the reaction rates are time-independent) or time-varying (if the reaction rates change over time). The work includes a theoretical characterization of the FSP truncation technique by showing that it can be put in the framework of inexact Krylov methods that relax matrix-vector products and compute them expediently by trading accuracy for speed. We also examine practical applications of our work in delay CME and parameter inference through local and global optimization schemes.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@book</span><span class="p">{</span><span class="nl">dinh2018inexact</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Inexact methods for the chemical master equation with constant or time-varying propensities, and application to parameter inference}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Dinh, Khanh N.}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{The University of Alabama}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2018}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> </ol> <h2 class="bibliography">2017</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/dinh2017analysis-480.webp 480w,/assets/img/publication_preview/dinh2017analysis-800.webp 800w,/assets/img/publication_preview/dinh2017analysis-1400.webp 1400w," type="image/webp" sizes="200px"></source> <img src="/assets/img/publication_preview/dinh2017analysis.jpg" class="preview z-depth-1 rounded" width="100%" height="auto" alt="dinh2017analysis.jpg" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="dinh2017analysis" class="col-sm-8"> <div class="title">Analysis of inexact Krylov subspace methods for approximating the matrix exponential</div> <div class="author"> <em>Khanh N. Dinh</em>, and <a href="https://math.ua.edu/people/roger-b-sidje/" rel="external nofollow noopener" target="_blank">Roger B. Sidje</a> </div> <div class="periodical"> <em>Mathematics and Computers in Simulation</em>, 2017 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1016/j.matcom.2017.01.002" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.sciencedirect.com/science/article/abs/pii/S0378475417300034?via%3Dihub" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/dinh2017analysis.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.matcom.2017.01.002" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Krylov subspace methods have proved quite effective at approximating the action of a large sparse matrix exponential on a vector. Their numerical robustness and matrix-free nature have enabled them to make inroads into a variety of applications. A case in point is solving the chemical master equation (CME) in the context of modeling biochemical reactions in biological cells. This is a challenging problem that gives rise to an extremely large matrix due to the curse of dimensionality. Inexact Krylov subspace methods that build on truncation techniques have helped solve some CME models that were considered computationally out of reach as recently as a few years ago. However, as models grow, truncating them means using an even smaller fraction of their whole extent, thereby introducing more inexactness. But experimental evidence suggests an apparent success and the aim of this study is to give theoretical insights into the reasons why. Essentially, we show that the truncation can be put in the framework of inexact Krylov methods that relax matrix–vector products and compute them expediently by trading accuracy for speed. This allows us to analyze both the residual (or defect) and the error of the resulting approximations to the matrix exponential from the viewpoint of inexact Krylov methods.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">dinh2017analysis</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Analysis of inexact Krylov subspace methods for approximating the matrix exponential}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Dinh, Khanh N. and Sidje, Roger B.}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Mathematics and Computers in Simulation}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{138}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1--13}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2017}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Elsevier}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.matcom.2017.01.002}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/dinh2017application-480.webp 480w,/assets/img/publication_preview/dinh2017application-800.webp 800w,/assets/img/publication_preview/dinh2017application-1400.webp 1400w," type="image/webp" sizes="200px"></source> <img src="/assets/img/publication_preview/dinh2017application.jpg" class="preview z-depth-1 rounded" width="100%" height="auto" alt="dinh2017application.jpg" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="dinh2017application" class="col-sm-8"> <div class="title">An application of the Krylov-FSP-SSA method to parameter fitting with maximum likelihood</div> <div class="author"> <em>Khanh N. Dinh</em>, and <a href="https://math.ua.edu/people/roger-b-sidje/" rel="external nofollow noopener" target="_blank">Roger B. Sidje</a> </div> <div class="periodical"> <em>Physical Biology</em>, 2017 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1088/1478-3975/aa868a" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://iopscience.iop.org/article/10.1088/1478-3975/aa868a" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/dinh2017application.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1088/1478-3975/aa868a" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Monte Carlo methods such as the stochastic simulation algorithm (SSA) have traditionally been employed in gene regulation problems. However, there has been increasing interest to directly obtain the probability distribution of the molecules involved by solving the chemical master equation (CME). This requires addressing the curse of dimensionality that is inherent in most gene regulation problems. The finite state projection (FSP) seeks to address the challenge and there have been variants that further reduce the size of the projection or that accelerate the resulting matrix exponential. The Krylov-FSP-SSA variant has proved numerically efficient by combining, on one hand, the SSA to adaptively drive the FSP, and on the other hand, adaptive Krylov techniques to evaluate the matrix exponential. Here we apply this Krylov-FSP-SSA to a mutual inhibitory gene network synthetically engineered in Saccharomyces cerevisiae, in which bimodality arises. We show numerically that the approach can efficiently approximate the transient probability distribution, and this has important implications for parameter fitting, where the CME has to be solved for many different parameter sets. The fitting scheme amounts to an optimization problem of finding the parameter set so that the transient probability distributions fit the observations with maximum likelihood. We compare five optimization schemes for this difficult problem, thereby providing further insights into this approach of parameter estimation that is often applied to models in systems biology where there is a need to calibrate free parameters.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">dinh2017application</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{An application of the Krylov-FSP-SSA method to parameter fitting with maximum likelihood}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Dinh, Khanh N. and Sidje, Roger B.}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Physical Biology}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{14}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{6}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{065001}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2017}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{IOP Publishing}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1088/1478-3975/aa868a}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/dinh2017adaptive-480.webp 480w,/assets/img/publication_preview/dinh2017adaptive-800.webp 800w,/assets/img/publication_preview/dinh2017adaptive-1400.webp 1400w," type="image/webp" sizes="200px"></source> <img src="/assets/img/publication_preview/dinh2017adaptive.jpg" class="preview z-depth-1 rounded" width="100%" height="auto" alt="dinh2017adaptive.jpg" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="dinh2017adaptive" class="col-sm-8"> <div class="title">An adaptive Magnus expansion method for solving the chemical master equation with time-dependent propensities</div> <div class="author"> <em>Khanh N. Dinh</em>, and <a href="https://math.ua.edu/people/roger-b-sidje/" rel="external nofollow noopener" target="_blank">Roger B. Sidje</a> </div> <div class="periodical"> <em>Journal of Coupled Systems and Multiscale Dynamics</em>, 2017 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1166/jcsmd.2017.1124" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.researchgate.net/publication/322007589_An_adaptive_Magnus_expansion_method_for_solving_the_chemical_master_equation_with_time-dependent_propensities" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/dinh2017adaptive.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1166/jcsmd.2017.1124" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>The chemical master equation (CME) is a system of ordinary differential equations (ODEs) to model the chemical interaction of molecular species. The largeness of the state space of the system makes solving the CME difficult, and this has motivated reduction strategies such as the finite state projection (FSP). Moreover, if the reaction rates are functions of the time, the CME becomes an ODE problem with time-dependent coefficients. Solution techniques include Monte Carlo algorithms, such as the stochastic simulation algorithm (SSA) or ODE solvers, such as Adams-PECE, Runge-Kutta and backward-differentiation formula (BDF). There are also Magnus-based solvers that have however not been thoroughly explored in the CME context. Here we introduce an adaptive time-stepping Magnus-SSA algorithm, in which the CME is solved using a Magnus expansion with not only a variable time-step but also with a variable state space that changes at each step via the SSA, and several error approximation approaches are attempted to monitor the adaptivity. We perform comparative tests against the classical Adams-PECE, Runge-Kutta and BDF methods on three biological problems, showing that the proposed adaptive Magnus-based variants can be efficient when the CME with time-dependent rates is stiff.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">dinh2017adaptive</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{An adaptive Magnus expansion method for solving the chemical master equation with time-dependent propensities}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Dinh, Khanh N. and Sidje, Roger B.}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Journal of Coupled Systems and Multiscale Dynamics}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{5}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{2}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{119--131}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2017}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{American Scientific Publishers}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1166/jcsmd.2017.1124}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> </ol> <h2 class="bibliography">2016</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/dinh2016understanding-480.webp 480w,/assets/img/publication_preview/dinh2016understanding-800.webp 800w,/assets/img/publication_preview/dinh2016understanding-1400.webp 1400w," type="image/webp" sizes="200px"></source> <img src="/assets/img/publication_preview/dinh2016understanding.jpg" class="preview z-depth-1 rounded" width="100%" height="auto" alt="dinh2016understanding.jpg" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="dinh2016understanding" class="col-sm-8"> <div class="title">Understanding the finite state projection and related methods for solving the chemical master equation</div> <div class="author"> <em>Khanh N. Dinh</em>, and <a href="https://math.ua.edu/people/roger-b-sidje/" rel="external nofollow noopener" target="_blank">Roger B. Sidje</a> </div> <div class="periodical"> <em>Physical Biology</em>, 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1088/1478-3975/13/3/035003" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://iopscience.iop.org/article/10.1088/1478-3975/13/3/035003" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/dinh2016understanding.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1088/1478-3975/13/3/035003" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>The finite state projection (FSP) method has enabled us to solve the chemical master equation of some biological models that were considered out of reach not long ago. Since the original FSP method, much effort has gone into transforming it into an adaptive time-stepping algorithm as well as studying its accuracy. Some of the improvements include the multiple time interval FSP, the sliding windows, and most notably the Krylov-FSP approach. Our goal in this tutorial is to give the reader an overview of the current methods that build on the FSP.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">dinh2016understanding</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Understanding the finite state projection and related methods for solving the chemical master equation}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Dinh, Khanh N. and Sidje, Roger B.}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Physical Biology}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{13}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{3}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{035003}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2016}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{IOP Publishing}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1088/1478-3975/13/3/035003}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li></ol> </div> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2026 Khanh N. Dinh. Last updated: January 30, 2026. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2781658a0a2b13ed609542042a859126"></script> <script defer src="/assets/js/common.js?e0514a05c5c95ac1a93a8dfd5249b92e"></script> <script defer src="/assets/js/copy_code.js?12775fdf7f95e901d7119054556e495f" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.2/es5/tex-mml-chtml.js" integrity="sha256-MASABpB4tYktI2Oitl4t+78w/lyA+D7b/s9GEP0JOGI=" crossorigin="anonymous"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6" crossorigin="anonymous"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> <script src="/assets/js/vanilla-back-to-top.min.js?f40d453793ff4f64e238e420181a1d17"></script> <script>addBackToTop();</script> </body> </html>